2015-06-09 12:01:36 Error: [PDOException] SQLSTATE[HY000]: General error: 8 attempt to write a readonly database
Request URL: /
Stack Trace:
#0 /var/www/html/app.florence.med.br/vendor/cakephp/cakephp/src/Database/Statement/StatementDecorator.php(169): PDOStatement->execute(NULL)
#1 /var/www/html/app.florence.med.br/vendor/cakephp/cakephp/src/Database/Statement/StatementDecorator.php(169): Cake\Database\Statement\StatementDecorator->execute(NULL)
#2 /var/www/html/app.florence.med.br/vendor/cakephp/cakephp/src/Database/Statement/BufferedStatement.php(74): Cake\Database\Statement\StatementDecorator->execute(NULL)
#3 /var/www/html/app.florence.med.br/vendor/cakephp/cakephp/src/Database/Statement/SqliteStatement.php(41): Cake\Database\Statement\BufferedStatement->execute(NULL)
#4 /var/www/html/app.florence.med.br/vendor/cakephp/cakephp/src/Database/Connection.php(274): Cake\Database\Statement\SqliteStatement->execute()
#5 /var/www/html/app.florence.med.br/vendor/cakephp/cakephp/src/Database/Query.php(175): Cake\Database\Connection->run(Object(Cake\ORM\Query))
#6 /var/www/html/app.florence.med.br/vendor/cakephp/cakephp/src/ORM/Table.php(1516): Cake\Database\Query->execute()
#7 /var/www/html/app.florence.med.br/vendor/cakephp/cakephp/src/ORM/Table.php(1436): Cake\ORM\Table->_insert(Object(Cake\ORM\Entity), Array)
#8 /var/www/html/app.florence.med.br/vendor/cakephp/cakephp/src/ORM/Table.php(1367): Cake\ORM\Table->_processSave(Object(Cake\ORM\Entity), Object(ArrayObject))
#9 /var/www/html/app.florence.med.br/vendor/cakephp/cakephp/src/Database/Connection.php(561): Cake\ORM\Table->Cake\ORM\{closure}(Object(Cake\Database\Connection))
#10 /var/www/html/app.florence.med.br/vendor/cakephp/cakephp/src/ORM/Table.php(1368): Cake\Database\Connection->transactional(Object(Closure))
#11 /var/www/html/app.florence.med.br/vendor/cakephp/debug_kit/src/Routing/Filter/DebugBarFilter.php(209): Cake\ORM\Table->save(Object(Cake\ORM\Entity))
#12 /var/www/html/app.florence.med.br/vendor/cakephp/cakephp/src/Event/EventManager.php(385): DebugKit\Routing\Filter\DebugBarFilter->afterDispatch(Object(Cake\Event\Event), Object(Cake\Network\Request), Object(Cake\Network\Response))
#13 /var/www/html/app.florence.med.br/vendor/cakephp/cakephp/src/Event/EventManager.php(351): Cake\Event\EventManager->_callListener(Array, Object(Cake\Event\Event))
#14 /var/www/html/app.florence.med.br/vendor/cakephp/cakephp/src/Event/EventManagerTrait.php(78): Cake\Event\EventManager->dispatch(Object(Cake\Event\Event))
#15 /var/www/html/app.florence.med.br/vendor/cakephp/cakephp/src/Routing/Dispatcher.php(92): Cake\Routing\Dispatcher->dispatchEvent('Dispatcher.afte...', Array)
#16 /var/www/html/app.florence.med.br/webroot/index.php(37): Cake\Routing\Dispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#17 {main}


2015-06-09 12:11:53 Error: [PDOException] SQLSTATE[HY000]: General error: 8 attempt to write a readonly database
Request URL: /
Stack Trace:
#0 /var/www/html/app.florence.med.br/vendor/cakephp/cakephp/src/Database/Statement/StatementDecorator.php(169): PDOStatement->execute(NULL)
#1 /var/www/html/app.florence.med.br/vendor/cakephp/cakephp/src/Database/Statement/StatementDecorator.php(169): Cake\Database\Statement\StatementDecorator->execute(NULL)
#2 /var/www/html/app.florence.med.br/vendor/cakephp/cakephp/src/Database/Statement/BufferedStatement.php(74): Cake\Database\Statement\StatementDecorator->execute(NULL)
#3 /var/www/html/app.florence.med.br/vendor/cakephp/cakephp/src/Database/Statement/SqliteStatement.php(41): Cake\Database\Statement\BufferedStatement->execute(NULL)
#4 /var/www/html/app.florence.med.br/vendor/cakephp/cakephp/src/Database/Connection.php(274): Cake\Database\Statement\SqliteStatement->execute()
#5 /var/www/html/app.florence.med.br/vendor/cakephp/cakephp/src/Database/Query.php(175): Cake\Database\Connection->run(Object(Cake\ORM\Query))
#6 /var/www/html/app.florence.med.br/vendor/cakephp/cakephp/src/ORM/Table.php(1516): Cake\Database\Query->execute()
#7 /var/www/html/app.florence.med.br/vendor/cakephp/cakephp/src/ORM/Table.php(1436): Cake\ORM\Table->_insert(Object(Cake\ORM\Entity), Array)
#8 /var/www/html/app.florence.med.br/vendor/cakephp/cakephp/src/ORM/Table.php(1367): Cake\ORM\Table->_processSave(Object(Cake\ORM\Entity), Object(ArrayObject))
#9 /var/www/html/app.florence.med.br/vendor/cakephp/cakephp/src/Database/Connection.php(561): Cake\ORM\Table->Cake\ORM\{closure}(Object(Cake\Database\Connection))
#10 /var/www/html/app.florence.med.br/vendor/cakephp/cakephp/src/ORM/Table.php(1368): Cake\Database\Connection->transactional(Object(Closure))
#11 /var/www/html/app.florence.med.br/vendor/cakephp/debug_kit/src/Routing/Filter/DebugBarFilter.php(209): Cake\ORM\Table->save(Object(Cake\ORM\Entity))
#12 /var/www/html/app.florence.med.br/vendor/cakephp/cakephp/src/Event/EventManager.php(385): DebugKit\Routing\Filter\DebugBarFilter->afterDispatch(Object(Cake\Event\Event), Object(Cake\Network\Request), Object(Cake\Network\Response))
#13 /var/www/html/app.florence.med.br/vendor/cakephp/cakephp/src/Event/EventManager.php(351): Cake\Event\EventManager->_callListener(Array, Object(Cake\Event\Event))
#14 /var/www/html/app.florence.med.br/vendor/cakephp/cakephp/src/Event/EventManagerTrait.php(78): Cake\Event\EventManager->dispatch(Object(Cake\Event\Event))
#15 /var/www/html/app.florence.med.br/vendor/cakephp/cakephp/src/Routing/Dispatcher.php(92): Cake\Routing\Dispatcher->dispatchEvent('Dispatcher.afte...', Array)
#16 /var/www/html/app.florence.med.br/webroot/index.php(37): Cake\Routing\Dispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#17 {main}


2015-06-09 12:11:54 Error: [PDOException] SQLSTATE[HY000]: General error: 8 attempt to write a readonly database
Request URL: /
Stack Trace:
#0 /var/www/html/app.florence.med.br/vendor/cakephp/cakephp/src/Database/Statement/StatementDecorator.php(169): PDOStatement->execute(NULL)
#1 /var/www/html/app.florence.med.br/vendor/cakephp/cakephp/src/Database/Statement/StatementDecorator.php(169): Cake\Database\Statement\StatementDecorator->execute(NULL)
#2 /var/www/html/app.florence.med.br/vendor/cakephp/cakephp/src/Database/Statement/BufferedStatement.php(74): Cake\Database\Statement\StatementDecorator->execute(NULL)
#3 /var/www/html/app.florence.med.br/vendor/cakephp/cakephp/src/Database/Statement/SqliteStatement.php(41): Cake\Database\Statement\BufferedStatement->execute(NULL)
#4 /var/www/html/app.florence.med.br/vendor/cakephp/cakephp/src/Database/Connection.php(274): Cake\Database\Statement\SqliteStatement->execute()
#5 /var/www/html/app.florence.med.br/vendor/cakephp/cakephp/src/Database/Query.php(175): Cake\Database\Connection->run(Object(Cake\ORM\Query))
#6 /var/www/html/app.florence.med.br/vendor/cakephp/cakephp/src/ORM/Table.php(1516): Cake\Database\Query->execute()
#7 /var/www/html/app.florence.med.br/vendor/cakephp/cakephp/src/ORM/Table.php(1436): Cake\ORM\Table->_insert(Object(Cake\ORM\Entity), Array)
#8 /var/www/html/app.florence.med.br/vendor/cakephp/cakephp/src/ORM/Table.php(1367): Cake\ORM\Table->_processSave(Object(Cake\ORM\Entity), Object(ArrayObject))
#9 /var/www/html/app.florence.med.br/vendor/cakephp/cakephp/src/Database/Connection.php(561): Cake\ORM\Table->Cake\ORM\{closure}(Object(Cake\Database\Connection))
#10 /var/www/html/app.florence.med.br/vendor/cakephp/cakephp/src/ORM/Table.php(1368): Cake\Database\Connection->transactional(Object(Closure))
#11 /var/www/html/app.florence.med.br/vendor/cakephp/debug_kit/src/Routing/Filter/DebugBarFilter.php(209): Cake\ORM\Table->save(Object(Cake\ORM\Entity))
#12 /var/www/html/app.florence.med.br/vendor/cakephp/cakephp/src/Event/EventManager.php(385): DebugKit\Routing\Filter\DebugBarFilter->afterDispatch(Object(Cake\Event\Event), Object(Cake\Network\Request), Object(Cake\Network\Response))
#13 /var/www/html/app.florence.med.br/vendor/cakephp/cakephp/src/Event/EventManager.php(351): Cake\Event\EventManager->_callListener(Array, Object(Cake\Event\Event))
#14 /var/www/html/app.florence.med.br/vendor/cakephp/cakephp/src/Event/EventManagerTrait.php(78): Cake\Event\EventManager->dispatch(Object(Cake\Event\Event))
#15 /var/www/html/app.florence.med.br/vendor/cakephp/cakephp/src/Routing/Dispatcher.php(92): Cake\Routing\Dispatcher->dispatchEvent('Dispatcher.afte...', Array)
#16 /var/www/html/app.florence.med.br/webroot/index.php(37): Cake\Routing\Dispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#17 {main}


2015-06-14 15:53:06 Error: Fatal Error (1): Maximum execution time of 30 seconds exceeded in [C:\Xampp\htdocs\florence-sistema\trunk\vendor\cakephp\cakephp\src\Database\Driver\PDODriverTrait.php, line 118]
Trace:
Cake\Error\BaseErrorHandler::handleFatalError() - CORE\src\Error\BaseErrorHandler.php, line 191
Cake\Error\BaseErrorHandler::Cake\Error\{closure}() - CORE\src\Error\BaseErrorHandler.php, line 90
[main] - [internal], line ??



2015-06-14 15:53:08 Error: [Cake\Error\FatalErrorException] Maximum execution time of 30 seconds exceeded
Request URL: /
Stack Trace:
#0 C:\Xampp\htdocs\florence-sistema\trunk\vendor\cakephp\cakephp\src\Error\BaseErrorHandler.php(90): Cake\Error\BaseErrorHandler->handleFatalError(1, 'Maximum executi...', 'C:\\Xampp\\htdocs...', 118)
#1 [internal function]: Cake\Error\BaseErrorHandler->Cake\Error\{closure}()
#2 {main}


2015-06-14 15:53:11 Error: [Cake\Routing\Exception\MissingControllerException] Controller class Favicon.ico could not be found.
Exception Attributes: array (
  'class' => 'Favicon.ico',
  'plugin' => NULL,
  'prefix' => NULL,
  '_ext' => NULL,
)
Request URL: /favicon.ico
Stack Trace:
#0 C:\Xampp\htdocs\florence-sistema\trunk\webroot\index.php(37): Cake\Routing\Dispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#1 {main}


2015-06-14 15:53:14 Error: [Cake\Routing\Exception\MissingControllerException] Controller class Favicon.ico could not be found.
Exception Attributes: array (
  'class' => 'Favicon.ico',
  'plugin' => NULL,
  'prefix' => NULL,
  '_ext' => NULL,
)
Request URL: /favicon.ico
Stack Trace:
#0 C:\Xampp\htdocs\florence-sistema\trunk\webroot\index.php(37): Cake\Routing\Dispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#1 {main}


2015-06-14 15:53:17 Error: [Cake\Routing\Exception\MissingControllerException] Controller class Favicon.ico could not be found.
Exception Attributes: array (
  'class' => 'Favicon.ico',
  'plugin' => NULL,
  'prefix' => NULL,
  '_ext' => NULL,
)
Request URL: /favicon.ico
Stack Trace:
#0 C:\Xampp\htdocs\florence-sistema\trunk\webroot\index.php(37): Cake\Routing\Dispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#1 {main}


2015-06-14 15:53:23 Error: [Cake\Routing\Exception\MissingControllerException] Controller class Favicon.ico could not be found.
Exception Attributes: array (
  'class' => 'Favicon.ico',
  'plugin' => NULL,
  'prefix' => NULL,
  '_ext' => NULL,
)
Request URL: /favicon.ico
Stack Trace:
#0 C:\Xampp\htdocs\florence-sistema\trunk\webroot\index.php(37): Cake\Routing\Dispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#1 {main}


2015-06-14 16:03:20 Error: Fatal Error (1): Call to undefined method App\Controller\ColaboradoresController::assign() in [C:\Xampp\htdocs\florence-sistema\trunk\src\Controller\ColaboradoresController.php, line 13]
Trace:
Cake\Error\BaseErrorHandler::handleFatalError() - CORE\src\Error\BaseErrorHandler.php, line 191
Cake\Error\BaseErrorHandler::Cake\Error\{closure}() - CORE\src\Error\BaseErrorHandler.php, line 90
[main] - [internal], line ??



2015-06-14 16:03:23 Error: [Cake\Error\FatalErrorException] Call to undefined method App\Controller\ColaboradoresController::assign()
Request URL: /
Stack Trace:
#0 C:\Xampp\htdocs\florence-sistema\trunk\vendor\cakephp\cakephp\src\Error\BaseErrorHandler.php(90): Cake\Error\BaseErrorHandler->handleFatalError(1, 'Call to undefin...', 'C:\\Xampp\\htdocs...', 13)
#1 [internal function]: Cake\Error\BaseErrorHandler->Cake\Error\{closure}()
#2 {main}


2015-06-14 16:03:25 Error: Fatal Error (1): Call to undefined method App\Controller\ColaboradoresController::assign() in [C:\Xampp\htdocs\florence-sistema\trunk\src\Controller\ColaboradoresController.php, line 13]
Trace:
Cake\Error\BaseErrorHandler::handleFatalError() - CORE\src\Error\BaseErrorHandler.php, line 191
Cake\Error\BaseErrorHandler::Cake\Error\{closure}() - CORE\src\Error\BaseErrorHandler.php, line 90
[main] - [internal], line ??



2015-06-14 16:03:26 Error: [Cake\Error\FatalErrorException] Call to undefined method App\Controller\ColaboradoresController::assign()
Request URL: /
Stack Trace:
#0 C:\Xampp\htdocs\florence-sistema\trunk\vendor\cakephp\cakephp\src\Error\BaseErrorHandler.php(90): Cake\Error\BaseErrorHandler->handleFatalError(1, 'Call to undefin...', 'C:\\Xampp\\htdocs...', 13)
#1 [internal function]: Cake\Error\BaseErrorHandler->Cake\Error\{closure}()
#2 {main}


2015-06-14 17:14:36 Error: [Cake\Routing\Exception\MissingControllerException] Controller class Agenda could not be found.
Exception Attributes: array (
  'class' => 'Agenda',
  'plugin' => NULL,
  'prefix' => NULL,
  '_ext' => NULL,
)
Request URL: /agenda
Stack Trace:
#0 C:\Xampp\htdocs\florence-sistema\trunk\webroot\index.php(37): Cake\Routing\Dispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#1 {main}


2015-06-14 17:15:05 Error: [Cake\Routing\Exception\MissingControllerException] Controller class Agenda could not be found.
Exception Attributes: array (
  'class' => 'Agenda',
  'plugin' => NULL,
  'prefix' => NULL,
  '_ext' => NULL,
)
Request URL: /agenda
Stack Trace:
#0 C:\Xampp\htdocs\florence-sistema\trunk\webroot\index.php(37): Cake\Routing\Dispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#1 {main}


2015-06-14 17:15:10 Error: [Cake\Routing\Exception\MissingControllerException] Controller class Agenda could not be found.
Exception Attributes: array (
  'class' => 'Agenda',
  'plugin' => NULL,
  'prefix' => NULL,
  '_ext' => NULL,
)
Request URL: /agenda
Stack Trace:
#0 C:\Xampp\htdocs\florence-sistema\trunk\webroot\index.php(37): Cake\Routing\Dispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#1 {main}


2015-06-14 17:15:41 Error: [Cake\Routing\Exception\MissingControllerException] Controller class Agenda could not be found.
Exception Attributes: array (
  'class' => 'Agenda',
  'plugin' => NULL,
  'prefix' => NULL,
  '_ext' => NULL,
)
Request URL: /agenda
Stack Trace:
#0 C:\Xampp\htdocs\florence-sistema\trunk\webroot\index.php(37): Cake\Routing\Dispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#1 {main}


2015-06-14 17:16:03 Error: [Cake\Routing\Exception\MissingControllerException] Controller class Agenda could not be found.
Exception Attributes: array (
  'class' => 'Agenda',
  'plugin' => NULL,
  'prefix' => NULL,
  '_ext' => NULL,
)
Request URL: /agenda
Stack Trace:
#0 C:\Xampp\htdocs\florence-sistema\trunk\webroot\index.php(37): Cake\Routing\Dispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#1 {main}


2015-06-14 17:16:50 Error: Fatal Error (4): syntax error, unexpected '}' in [C:\Xampp\htdocs\florence-sistema\trunk\src\Template\Agendas\index.ctp, line 9]
Trace:
Cake\Error\BaseErrorHandler::handleFatalError() - CORE\src\Error\BaseErrorHandler.php, line 191
Cake\Error\BaseErrorHandler::Cake\Error\{closure}() - CORE\src\Error\BaseErrorHandler.php, line 90
[main] - [internal], line ??



2015-06-14 17:16:50 Error: [Cake\Error\FatalErrorException] syntax error, unexpected '}'
Request URL: /agendas
Stack Trace:
#0 C:\Xampp\htdocs\florence-sistema\trunk\vendor\cakephp\cakephp\src\Error\BaseErrorHandler.php(90): Cake\Error\BaseErrorHandler->handleFatalError(4, 'syntax error, u...', 'C:\\Xampp\\htdocs...', 9)
#1 [internal function]: Cake\Error\BaseErrorHandler->Cake\Error\{closure}()
#2 {main}


2015-06-14 17:18:11 Error: Fatal Error (4): syntax error, unexpected 'weekends' (T_STRING) in [C:\Xampp\htdocs\florence-sistema\trunk\src\Template\Agendas\index.ctp, line 6]
Trace:
Cake\Error\BaseErrorHandler::handleFatalError() - CORE\src\Error\BaseErrorHandler.php, line 191
Cake\Error\BaseErrorHandler::Cake\Error\{closure}() - CORE\src\Error\BaseErrorHandler.php, line 90
[main] - [internal], line ??



2015-06-14 17:18:12 Error: [Cake\Error\FatalErrorException] syntax error, unexpected 'weekends' (T_STRING)
Request URL: /agendas
Stack Trace:
#0 C:\Xampp\htdocs\florence-sistema\trunk\vendor\cakephp\cakephp\src\Error\BaseErrorHandler.php(90): Cake\Error\BaseErrorHandler->handleFatalError(4, 'syntax error, u...', 'C:\\Xampp\\htdocs...', 6)
#1 [internal function]: Cake\Error\BaseErrorHandler->Cake\Error\{closure}()
#2 {main}


2015-06-14 17:19:06 Error: Fatal Error (4): syntax error, unexpected 'weekends' (T_STRING) in [C:\Xampp\htdocs\florence-sistema\trunk\src\Template\Agendas\index.ctp, line 6]
Trace:
Cake\Error\BaseErrorHandler::handleFatalError() - CORE\src\Error\BaseErrorHandler.php, line 191
Cake\Error\BaseErrorHandler::Cake\Error\{closure}() - CORE\src\Error\BaseErrorHandler.php, line 90
[main] - [internal], line ??



2015-06-14 17:19:07 Error: [Cake\Error\FatalErrorException] syntax error, unexpected 'weekends' (T_STRING)
Request URL: /agendas
Stack Trace:
#0 C:\Xampp\htdocs\florence-sistema\trunk\vendor\cakephp\cakephp\src\Error\BaseErrorHandler.php(90): Cake\Error\BaseErrorHandler->handleFatalError(4, 'syntax error, u...', 'C:\\Xampp\\htdocs...', 6)
#1 [internal function]: Cake\Error\BaseErrorHandler->Cake\Error\{closure}()
#2 {main}


2015-06-14 17:19:26 Error: Fatal Error (4): syntax error, unexpected 'weekends' (T_STRING) in [C:\Xampp\htdocs\florence-sistema\trunk\src\Template\Agendas\index.ctp, line 8]
Trace:
Cake\Error\BaseErrorHandler::handleFatalError() - CORE\src\Error\BaseErrorHandler.php, line 191
Cake\Error\BaseErrorHandler::Cake\Error\{closure}() - CORE\src\Error\BaseErrorHandler.php, line 90
[main] - [internal], line ??



2015-06-14 17:19:27 Error: [Cake\Error\FatalErrorException] syntax error, unexpected 'weekends' (T_STRING)
Request URL: /agendas
Stack Trace:
#0 C:\Xampp\htdocs\florence-sistema\trunk\vendor\cakephp\cakephp\src\Error\BaseErrorHandler.php(90): Cake\Error\BaseErrorHandler->handleFatalError(4, 'syntax error, u...', 'C:\\Xampp\\htdocs...', 8)
#1 [internal function]: Cake\Error\BaseErrorHandler->Cake\Error\{closure}()
#2 {main}


2015-06-14 17:20:11 Error: Fatal Error (4): syntax error, unexpected '}' in [C:\Xampp\htdocs\florence-sistema\trunk\src\Template\Agendas\index.ctp, line 9]
Trace:
Cake\Error\BaseErrorHandler::handleFatalError() - CORE\src\Error\BaseErrorHandler.php, line 191
Cake\Error\BaseErrorHandler::Cake\Error\{closure}() - CORE\src\Error\BaseErrorHandler.php, line 90
[main] - [internal], line ??



2015-06-14 17:20:12 Error: [Cake\Error\FatalErrorException] syntax error, unexpected '}'
Request URL: /agendas
Stack Trace:
#0 C:\Xampp\htdocs\florence-sistema\trunk\vendor\cakephp\cakephp\src\Error\BaseErrorHandler.php(90): Cake\Error\BaseErrorHandler->handleFatalError(4, 'syntax error, u...', 'C:\\Xampp\\htdocs...', 9)
#1 [internal function]: Cake\Error\BaseErrorHandler->Cake\Error\{closure}()
#2 {main}


2015-06-14 17:21:52 Error: Fatal Error (4): syntax error, unexpected '}' in [C:\Xampp\htdocs\florence-sistema\trunk\src\Template\Agendas\index.ctp, line 5]
Trace:
Cake\Error\BaseErrorHandler::handleFatalError() - CORE\src\Error\BaseErrorHandler.php, line 191
Cake\Error\BaseErrorHandler::Cake\Error\{closure}() - CORE\src\Error\BaseErrorHandler.php, line 90
[main] - [internal], line ??



2015-06-14 17:21:53 Error: [Cake\Error\FatalErrorException] syntax error, unexpected '}'
Request URL: /agendas
Stack Trace:
#0 C:\Xampp\htdocs\florence-sistema\trunk\vendor\cakephp\cakephp\src\Error\BaseErrorHandler.php(90): Cake\Error\BaseErrorHandler->handleFatalError(4, 'syntax error, u...', 'C:\\Xampp\\htdocs...', 5)
#1 [internal function]: Cake\Error\BaseErrorHandler->Cake\Error\{closure}()
#2 {main}


2015-06-14 19:46:30 Error: [Cake\View\Exception\MissingTemplateException] Template file "Agendas\adicionar.ctp" is missing.
Exception Attributes: array (
  'file' => 'Agendas\\adicionar.ctp',
)
Request URL: /agendas/adicionar/15-06-2015%2006:00/1
Stack Trace:
#0 C:\Xampp\htdocs\florence-sistema\trunk\vendor\cakephp\cakephp\src\View\View.php(460): Cake\View\View->_getViewFileName(NULL)
#1 C:\Xampp\htdocs\florence-sistema\trunk\vendor\cakephp\cakephp\src\Controller\Controller.php(575): Cake\View\View->render(NULL, NULL)
#2 C:\Xampp\htdocs\florence-sistema\trunk\vendor\cakephp\cakephp\src\Routing\Dispatcher.php(120): Cake\Controller\Controller->render()
#3 C:\Xampp\htdocs\florence-sistema\trunk\vendor\cakephp\cakephp\src\Routing\Dispatcher.php(87): Cake\Routing\Dispatcher->_invoke(Object(App\Controller\AgendasController))
#4 C:\Xampp\htdocs\florence-sistema\trunk\webroot\index.php(37): Cake\Routing\Dispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#5 {main}


2015-06-14 19:46:31 Error: [Cake\Routing\Exception\MissingControllerException] Controller class Favicon.ico could not be found.
Exception Attributes: array (
  'class' => 'Favicon.ico',
  'plugin' => NULL,
  'prefix' => NULL,
  '_ext' => NULL,
)
Request URL: /favicon.ico
Stack Trace:
#0 C:\Xampp\htdocs\florence-sistema\trunk\webroot\index.php(37): Cake\Routing\Dispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#1 {main}


2015-06-14 20:23:24 Error: Fatal Error (4): syntax error, unexpected ',' in [C:\Xampp\htdocs\florence-sistema\trunk\src\Template\Agendas\adicionar.ctp, line 7]
Trace:
Cake\Error\BaseErrorHandler::handleFatalError() - CORE\src\Error\BaseErrorHandler.php, line 191
Cake\Error\BaseErrorHandler::Cake\Error\{closure}() - CORE\src\Error\BaseErrorHandler.php, line 90
[main] - [internal], line ??



2015-06-14 20:23:25 Error: [Cake\Error\FatalErrorException] syntax error, unexpected ','
Request URL: /agendas/adicionar/15-06-2015%2006:00/1
Stack Trace:
#0 C:\Xampp\htdocs\florence-sistema\trunk\vendor\cakephp\cakephp\src\Error\BaseErrorHandler.php(90): Cake\Error\BaseErrorHandler->handleFatalError(4, 'syntax error, u...', 'C:\\Xampp\\htdocs...', 7)
#1 [internal function]: Cake\Error\BaseErrorHandler->Cake\Error\{closure}()
#2 {main}


2015-06-14 20:25:54 Error: Fatal Error (4): syntax error, unexpected '').on('' (T_CONSTANT_ENCAPSED_STRING) in [C:\Xampp\htdocs\florence-sistema\trunk\src\Template\Agendas\adicionar.ctp, line 5]
Trace:
Cake\Error\BaseErrorHandler::handleFatalError() - CORE\src\Error\BaseErrorHandler.php, line 191
Cake\Error\BaseErrorHandler::Cake\Error\{closure}() - CORE\src\Error\BaseErrorHandler.php, line 90
[main] - [internal], line ??



2015-06-14 20:25:55 Error: [Cake\Error\FatalErrorException] syntax error, unexpected '').on('' (T_CONSTANT_ENCAPSED_STRING)
Request URL: /agendas/adicionar/15-06-2015%2006:00/1
Stack Trace:
#0 C:\Xampp\htdocs\florence-sistema\trunk\vendor\cakephp\cakephp\src\Error\BaseErrorHandler.php(90): Cake\Error\BaseErrorHandler->handleFatalError(4, 'syntax error, u...', 'C:\\Xampp\\htdocs...', 5)
#1 [internal function]: Cake\Error\BaseErrorHandler->Cake\Error\{closure}()
#2 {main}


2015-06-14 20:26:10 Error: Fatal Error (4): syntax error, unexpected '').on('' (T_CONSTANT_ENCAPSED_STRING) in [C:\Xampp\htdocs\florence-sistema\trunk\src\Template\Agendas\adicionar.ctp, line 5]
Trace:
Cake\Error\BaseErrorHandler::handleFatalError() - CORE\src\Error\BaseErrorHandler.php, line 191
Cake\Error\BaseErrorHandler::Cake\Error\{closure}() - CORE\src\Error\BaseErrorHandler.php, line 90
[main] - [internal], line ??



2015-06-14 20:26:10 Error: [Cake\Error\FatalErrorException] syntax error, unexpected '').on('' (T_CONSTANT_ENCAPSED_STRING)
Request URL: /agendas/adicionar/15-06-2015%2006:00/1
Stack Trace:
#0 C:\Xampp\htdocs\florence-sistema\trunk\vendor\cakephp\cakephp\src\Error\BaseErrorHandler.php(90): Cake\Error\BaseErrorHandler->handleFatalError(4, 'syntax error, u...', 'C:\\Xampp\\htdocs...', 5)
#1 [internal function]: Cake\Error\BaseErrorHandler->Cake\Error\{closure}()
#2 {main}


2015-06-14 20:26:27 Error: Fatal Error (4): syntax error, unexpected '').on('' (T_CONSTANT_ENCAPSED_STRING) in [C:\Xampp\htdocs\florence-sistema\trunk\src\Template\Agendas\adicionar.ctp, line 5]
Trace:
Cake\Error\BaseErrorHandler::handleFatalError() - CORE\src\Error\BaseErrorHandler.php, line 191
Cake\Error\BaseErrorHandler::Cake\Error\{closure}() - CORE\src\Error\BaseErrorHandler.php, line 90
[main] - [internal], line ??



2015-06-14 20:26:28 Error: [Cake\Error\FatalErrorException] syntax error, unexpected '').on('' (T_CONSTANT_ENCAPSED_STRING)
Request URL: /agendas/adicionar/15-06-2015%2006:00/1
Stack Trace:
#0 C:\Xampp\htdocs\florence-sistema\trunk\vendor\cakephp\cakephp\src\Error\BaseErrorHandler.php(90): Cake\Error\BaseErrorHandler->handleFatalError(4, 'syntax error, u...', 'C:\\Xampp\\htdocs...', 5)
#1 [internal function]: Cake\Error\BaseErrorHandler->Cake\Error\{closure}()
#2 {main}


2015-06-14 20:27:13 Error: Fatal Error (4): syntax error, unexpected '').on('' (T_CONSTANT_ENCAPSED_STRING) in [C:\Xampp\htdocs\florence-sistema\trunk\src\Template\Agendas\adicionar.ctp, line 5]
Trace:
Cake\Error\BaseErrorHandler::handleFatalError() - CORE\src\Error\BaseErrorHandler.php, line 191
Cake\Error\BaseErrorHandler::Cake\Error\{closure}() - CORE\src\Error\BaseErrorHandler.php, line 90
[main] - [internal], line ??



2015-06-14 20:27:13 Error: [Cake\Error\FatalErrorException] syntax error, unexpected '').on('' (T_CONSTANT_ENCAPSED_STRING)
Request URL: /agendas/adicionar/15-06-2015%2006:00/1
Stack Trace:
#0 C:\Xampp\htdocs\florence-sistema\trunk\vendor\cakephp\cakephp\src\Error\BaseErrorHandler.php(90): Cake\Error\BaseErrorHandler->handleFatalError(4, 'syntax error, u...', 'C:\\Xampp\\htdocs...', 5)
#1 [internal function]: Cake\Error\BaseErrorHandler->Cake\Error\{closure}()
#2 {main}


2015-06-14 20:27:19 Error: Fatal Error (4): syntax error, unexpected '').on('' (T_CONSTANT_ENCAPSED_STRING) in [C:\Xampp\htdocs\florence-sistema\trunk\src\Template\Agendas\adicionar.ctp, line 5]
Trace:
Cake\Error\BaseErrorHandler::handleFatalError() - CORE\src\Error\BaseErrorHandler.php, line 191
Cake\Error\BaseErrorHandler::Cake\Error\{closure}() - CORE\src\Error\BaseErrorHandler.php, line 90
[main] - [internal], line ??



2015-06-14 20:27:20 Error: [Cake\Error\FatalErrorException] syntax error, unexpected '').on('' (T_CONSTANT_ENCAPSED_STRING)
Request URL: /agendas/adicionar/15-06-2015%2006:00/1
Stack Trace:
#0 C:\Xampp\htdocs\florence-sistema\trunk\vendor\cakephp\cakephp\src\Error\BaseErrorHandler.php(90): Cake\Error\BaseErrorHandler->handleFatalError(4, 'syntax error, u...', 'C:\\Xampp\\htdocs...', 5)
#1 [internal function]: Cake\Error\BaseErrorHandler->Cake\Error\{closure}()
#2 {main}


2015-06-14 22:42:24 Error: [Cake\Controller\Exception\MissingActionException] Action PacientesController::adicionar-ajax() could not be found, or is not accessible.
Exception Attributes: array (
  'controller' => 'PacientesController',
  'action' => 'adicionar-ajax',
  'prefix' => '',
  'plugin' => NULL,
)
Request URL: /pacientes/adicionar-ajax
Stack Trace:
#0 C:\Xampp\htdocs\florence-sistema\trunk\vendor\cakephp\cakephp\src\Routing\Dispatcher.php(114): Cake\Controller\Controller->invokeAction()
#1 C:\Xampp\htdocs\florence-sistema\trunk\vendor\cakephp\cakephp\src\Routing\Dispatcher.php(87): Cake\Routing\Dispatcher->_invoke(Object(App\Controller\PacientesController))
#2 C:\Xampp\htdocs\florence-sistema\trunk\webroot\index.php(37): Cake\Routing\Dispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#3 {main}


2015-06-14 22:50:54 Error: [Cake\View\Exception\MissingTemplateException] Template file "Pacientes\adicionar_modal.ctp" is missing.
Exception Attributes: array (
  'file' => 'Pacientes\\adicionar_modal.ctp',
)
Request URL: /pacientes/adicionar/adicionar-modal/
Stack Trace:
#0 C:\Xampp\htdocs\florence-sistema\trunk\vendor\cakephp\cakephp\src\View\View.php(460): Cake\View\View->_getViewFileName('adicionar-modal')
#1 C:\Xampp\htdocs\florence-sistema\trunk\vendor\cakephp\cakephp\src\Controller\Controller.php(575): Cake\View\View->render('adicionar-modal', NULL)
#2 C:\Xampp\htdocs\florence-sistema\trunk\src\Controller\PacientesController.php(20): Cake\Controller\Controller->render('adicionar-modal')
#3 [internal function]: App\Controller\PacientesController->adicionar('adicionar-modal')
#4 C:\Xampp\htdocs\florence-sistema\trunk\vendor\cakephp\cakephp\src\Controller\Controller.php(404): call_user_func_array(Array, Array)
#5 C:\Xampp\htdocs\florence-sistema\trunk\vendor\cakephp\cakephp\src\Routing\Dispatcher.php(114): Cake\Controller\Controller->invokeAction()
#6 C:\Xampp\htdocs\florence-sistema\trunk\vendor\cakephp\cakephp\src\Routing\Dispatcher.php(87): Cake\Routing\Dispatcher->_invoke(Object(App\Controller\PacientesController))
#7 C:\Xampp\htdocs\florence-sistema\trunk\webroot\index.php(37): Cake\Routing\Dispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#8 {main}


2015-06-14 23:06:48 Error: [Cake\Routing\Exception\MissingControllerException] Controller class Favicon.ico could not be found.
Exception Attributes: array (
  'class' => 'Favicon.ico',
  'plugin' => NULL,
  'prefix' => NULL,
  '_ext' => NULL,
)
Request URL: /favicon.ico
Stack Trace:
#0 C:\Xampp\htdocs\florence-sistema\trunk\webroot\index.php(37): Cake\Routing\Dispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#1 {main}


2015-06-14 23:07:54 Error: [Cake\Controller\Exception\MissingActionException] Action PacientesController::adicionar_modal() could not be found, or is not accessible.
Exception Attributes: array (
  'controller' => 'PacientesController',
  'action' => 'adicionar_modal',
  'prefix' => '',
  'plugin' => NULL,
)
Request URL: /pacientes/adicionar_modal/
Stack Trace:
#0 C:\Xampp\htdocs\florence-sistema\trunk\vendor\cakephp\cakephp\src\Routing\Dispatcher.php(114): Cake\Controller\Controller->invokeAction()
#1 C:\Xampp\htdocs\florence-sistema\trunk\vendor\cakephp\cakephp\src\Routing\Dispatcher.php(87): Cake\Routing\Dispatcher->_invoke(Object(App\Controller\PacientesController))
#2 C:\Xampp\htdocs\florence-sistema\trunk\webroot\index.php(37): Cake\Routing\Dispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#3 {main}


2015-06-14 23:37:52 Error: [Cake\View\Exception\MissingHelperException] Helper class paramsHelper could not be found.
Exception Attributes: array (
  'class' => 'paramsHelper',
  'plugin' => NULL,
)
Request URL: /agendas/adicionar/15-06-2015%2006:00/3
Stack Trace:
#0 C:\Xampp\htdocs\florence-sistema\trunk\vendor\cakephp\cakephp\src\Core\ObjectRegistry.php(86): Cake\View\HelperRegistry->_throwMissingClassError('params', NULL)
#1 C:\Xampp\htdocs\florence-sistema\trunk\vendor\cakephp\cakephp\src\View\HelperRegistry.php(66): Cake\Core\ObjectRegistry->load('params')
#2 C:\Xampp\htdocs\florence-sistema\trunk\vendor\cakephp\cakephp\src\View\View.php(732): Cake\View\HelperRegistry->__isset('params')
#3 C:\Xampp\htdocs\florence-sistema\trunk\src\Template\Agendas\adicionar.ctp(23): Cake\View\View->__get('params')
#4 C:\Xampp\htdocs\florence-sistema\trunk\vendor\cakephp\cakephp\src\View\View.php(815): include('C:\\Xampp\\htdocs...')
#5 C:\Xampp\htdocs\florence-sistema\trunk\vendor\cakephp\cakephp\src\View\View.php(775): Cake\View\View->_evaluate('C:\\Xampp\\htdocs...', Array)
#6 C:\Xampp\htdocs\florence-sistema\trunk\vendor\cakephp\cakephp\src\View\View.php(463): Cake\View\View->_render('C:\\Xampp\\htdocs...')
#7 C:\Xampp\htdocs\florence-sistema\trunk\vendor\cakephp\cakephp\src\Controller\Controller.php(575): Cake\View\View->render(NULL, NULL)
#8 C:\Xampp\htdocs\florence-sistema\trunk\vendor\cakephp\cakephp\src\Routing\Dispatcher.php(120): Cake\Controller\Controller->render()
#9 C:\Xampp\htdocs\florence-sistema\trunk\vendor\cakephp\cakephp\src\Routing\Dispatcher.php(87): Cake\Routing\Dispatcher->_invoke(Object(App\Controller\AgendasController))
#10 C:\Xampp\htdocs\florence-sistema\trunk\webroot\index.php(37): Cake\Routing\Dispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#11 {main}


2015-06-14 23:48:23 Error: Fatal Error (4): syntax error, unexpected '=>' (T_DOUBLE_ARROW) in [C:\Xampp\htdocs\florence-sistema\trunk\src\Template\Agendas\adicionar.ctp, line 37]
Trace:
Cake\Error\BaseErrorHandler::handleFatalError() - CORE\src\Error\BaseErrorHandler.php, line 191
Cake\Error\BaseErrorHandler::Cake\Error\{closure}() - CORE\src\Error\BaseErrorHandler.php, line 90
[main] - [internal], line ??



2015-06-14 23:48:24 Error: [Cake\Error\FatalErrorException] syntax error, unexpected '=>' (T_DOUBLE_ARROW)
Request URL: /agendas/adicionar/15-06-2015%2006:00/3
Stack Trace:
#0 C:\Xampp\htdocs\florence-sistema\trunk\vendor\cakephp\cakephp\src\Error\BaseErrorHandler.php(90): Cake\Error\BaseErrorHandler->handleFatalError(4, 'syntax error, u...', 'C:\\Xampp\\htdocs...', 37)
#1 [internal function]: Cake\Error\BaseErrorHandler->Cake\Error\{closure}()
#2 {main}


2015-06-15 01:30:09 Error: [Cake\View\Exception\MissingHelperException] Helper class passwordHelper could not be found.
Exception Attributes: array (
  'class' => 'passwordHelper',
  'plugin' => NULL,
)
Request URL: /colaboradores/adicionar
Stack Trace:
#0 C:\Xampp\htdocs\florence-sistema\trunk\vendor\cakephp\cakephp\src\Core\ObjectRegistry.php(86): Cake\View\HelperRegistry->_throwMissingClassError('password', NULL)
#1 C:\Xampp\htdocs\florence-sistema\trunk\vendor\cakephp\cakephp\src\View\HelperRegistry.php(66): Cake\Core\ObjectRegistry->load('password')
#2 C:\Xampp\htdocs\florence-sistema\trunk\vendor\cakephp\cakephp\src\View\View.php(732): Cake\View\HelperRegistry->__isset('password')
#3 C:\Xampp\htdocs\florence-sistema\trunk\src\Template\Colaboradores\adicionar.ctp(35): Cake\View\View->__get('password')
#4 C:\Xampp\htdocs\florence-sistema\trunk\vendor\cakephp\cakephp\src\View\View.php(815): include('C:\\Xampp\\htdocs...')
#5 C:\Xampp\htdocs\florence-sistema\trunk\vendor\cakephp\cakephp\src\View\View.php(775): Cake\View\View->_evaluate('C:\\Xampp\\htdocs...', Array)
#6 C:\Xampp\htdocs\florence-sistema\trunk\vendor\cakephp\cakephp\src\View\View.php(463): Cake\View\View->_render('C:\\Xampp\\htdocs...')
#7 C:\Xampp\htdocs\florence-sistema\trunk\vendor\cakephp\cakephp\src\Controller\Controller.php(575): Cake\View\View->render(NULL, NULL)
#8 C:\Xampp\htdocs\florence-sistema\trunk\vendor\cakephp\cakephp\src\Routing\Dispatcher.php(120): Cake\Controller\Controller->render()
#9 C:\Xampp\htdocs\florence-sistema\trunk\vendor\cakephp\cakephp\src\Routing\Dispatcher.php(87): Cake\Routing\Dispatcher->_invoke(Object(App\Controller\ColaboradoresController))
#10 C:\Xampp\htdocs\florence-sistema\trunk\webroot\index.php(37): Cake\Routing\Dispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#11 {main}


2015-06-15 01:31:52 Error: Fatal Error (4): syntax error, unexpected '=>' (T_DOUBLE_ARROW) in [C:\Xampp\htdocs\florence-sistema\trunk\src\Template\Colaboradores\adicionar.ctp, line 62]
Trace:
Cake\Error\BaseErrorHandler::handleFatalError() - CORE\src\Error\BaseErrorHandler.php, line 191
Cake\Error\BaseErrorHandler::Cake\Error\{closure}() - CORE\src\Error\BaseErrorHandler.php, line 90
[main] - [internal], line ??



2015-06-15 01:31:52 Error: [Cake\Error\FatalErrorException] syntax error, unexpected '=>' (T_DOUBLE_ARROW)
Request URL: /colaboradores/adicionar
Stack Trace:
#0 C:\Xampp\htdocs\florence-sistema\trunk\vendor\cakephp\cakephp\src\Error\BaseErrorHandler.php(90): Cake\Error\BaseErrorHandler->handleFatalError(4, 'syntax error, u...', 'C:\\Xampp\\htdocs...', 62)
#1 [internal function]: Cake\Error\BaseErrorHandler->Cake\Error\{closure}()
#2 {main}


2015-06-16 23:05:17 Error: Fatal Error (4): syntax error, unexpected ''2'' (T_CONSTANT_ENCAPSED_STRING), expecting ']' in [C:\Xampp\htdocs\florence-sistema\trunk\src\Template\Permissoes\adicionar.ctp, line 45]
Trace:
Cake\Error\BaseErrorHandler::handleFatalError() - CORE\src\Error\BaseErrorHandler.php, line 191
Cake\Error\BaseErrorHandler::Cake\Error\{closure}() - CORE\src\Error\BaseErrorHandler.php, line 90
[main] - [internal], line ??



2015-06-16 23:05:18 Error: [Cake\Error\FatalErrorException] syntax error, unexpected ''2'' (T_CONSTANT_ENCAPSED_STRING), expecting ']'
Request URL: /permissoes/adicionar
Stack Trace:
#0 C:\Xampp\htdocs\florence-sistema\trunk\vendor\cakephp\cakephp\src\Error\BaseErrorHandler.php(90): Cake\Error\BaseErrorHandler->handleFatalError(4, 'syntax error, u...', 'C:\\Xampp\\htdocs...', 45)
#1 [internal function]: Cake\Error\BaseErrorHandler->Cake\Error\{closure}()
#2 {main}


2015-06-16 23:05:45 Error: Fatal Error (4): syntax error, unexpected 'PermissaoPermissoesId' (T_STRING), expecting ']' in [C:\Xampp\htdocs\florence-sistema\trunk\src\Template\Permissoes\adicionar.ctp, line 53]
Trace:
Cake\Error\BaseErrorHandler::handleFatalError() - CORE\src\Error\BaseErrorHandler.php, line 191
Cake\Error\BaseErrorHandler::Cake\Error\{closure}() - CORE\src\Error\BaseErrorHandler.php, line 90
[main] - [internal], line ??



2015-06-16 23:05:46 Error: [Cake\Error\FatalErrorException] syntax error, unexpected 'PermissaoPermissoesId' (T_STRING), expecting ']'
Request URL: /permissoes/adicionar
Stack Trace:
#0 C:\Xampp\htdocs\florence-sistema\trunk\vendor\cakephp\cakephp\src\Error\BaseErrorHandler.php(90): Cake\Error\BaseErrorHandler->handleFatalError(4, 'syntax error, u...', 'C:\\Xampp\\htdocs...', 53)
#1 [internal function]: Cake\Error\BaseErrorHandler->Cake\Error\{closure}()
#2 {main}


2015-06-16 23:07:08 Error: Fatal Error (4): syntax error, unexpected 'PermissaoPermissoesId' (T_STRING), expecting ']' in [C:\Xampp\htdocs\florence-sistema\trunk\src\Template\Permissoes\adicionar.ctp, line 43]
Trace:
Cake\Error\BaseErrorHandler::handleFatalError() - CORE\src\Error\BaseErrorHandler.php, line 191
Cake\Error\BaseErrorHandler::Cake\Error\{closure}() - CORE\src\Error\BaseErrorHandler.php, line 90
[main] - [internal], line ??



2015-06-16 23:07:09 Error: [Cake\Error\FatalErrorException] syntax error, unexpected 'PermissaoPermissoesId' (T_STRING), expecting ']'
Request URL: /permissoes/adicionar
Stack Trace:
#0 C:\Xampp\htdocs\florence-sistema\trunk\vendor\cakephp\cakephp\src\Error\BaseErrorHandler.php(90): Cake\Error\BaseErrorHandler->handleFatalError(4, 'syntax error, u...', 'C:\\Xampp\\htdocs...', 43)
#1 [internal function]: Cake\Error\BaseErrorHandler->Cake\Error\{closure}()
#2 {main}


2015-06-16 23:07:40 Error: Fatal Error (4): syntax error, unexpected 'PermissaoPermissoesId' (T_STRING), expecting ']' in [C:\Xampp\htdocs\florence-sistema\trunk\src\Template\Permissoes\adicionar.ctp, line 43]
Trace:
Cake\Error\BaseErrorHandler::handleFatalError() - CORE\src\Error\BaseErrorHandler.php, line 191
Cake\Error\BaseErrorHandler::Cake\Error\{closure}() - CORE\src\Error\BaseErrorHandler.php, line 90
[main] - [internal], line ??



2015-06-16 23:07:41 Error: [Cake\Error\FatalErrorException] syntax error, unexpected 'PermissaoPermissoesId' (T_STRING), expecting ']'
Request URL: /permissoes/adicionar
Stack Trace:
#0 C:\Xampp\htdocs\florence-sistema\trunk\vendor\cakephp\cakephp\src\Error\BaseErrorHandler.php(90): Cake\Error\BaseErrorHandler->handleFatalError(4, 'syntax error, u...', 'C:\\Xampp\\htdocs...', 43)
#1 [internal function]: Cake\Error\BaseErrorHandler->Cake\Error\{closure}()
#2 {main}


2015-06-17 00:12:02 Error: Fatal Error (4): syntax error, unexpected '2' (T_LNUMBER), expecting ']' in [C:\Xampp\htdocs\florence-sistema\trunk\src\Template\Grupos\adicionar.ctp, line 38]
Trace:
Cake\Error\BaseErrorHandler::handleFatalError() - CORE\src\Error\BaseErrorHandler.php, line 191
Cake\Error\BaseErrorHandler::Cake\Error\{closure}() - CORE\src\Error\BaseErrorHandler.php, line 90
[main] - [internal], line ??



2015-06-17 00:12:03 Error: [Cake\Error\FatalErrorException] syntax error, unexpected '2' (T_LNUMBER), expecting ']'
Request URL: /grupos/adicionar
Stack Trace:
#0 C:\Xampp\htdocs\florence-sistema\trunk\vendor\cakephp\cakephp\src\Error\BaseErrorHandler.php(90): Cake\Error\BaseErrorHandler->handleFatalError(4, 'syntax error, u...', 'C:\\Xampp\\htdocs...', 38)
#1 [internal function]: Cake\Error\BaseErrorHandler->Cake\Error\{closure}()
#2 {main}


2015-06-17 00:23:11 Error: Fatal Error (4): syntax error, unexpected 'de' (T_STRING), expecting ']' in [C:\Xampp\htdocs\florence-sistema\trunk\src\Template\Grupos\adicionar.ctp, line 35]
Trace:
Cake\Error\BaseErrorHandler::handleFatalError() - CORE\src\Error\BaseErrorHandler.php, line 191
Cake\Error\BaseErrorHandler::Cake\Error\{closure}() - CORE\src\Error\BaseErrorHandler.php, line 90
[main] - [internal], line ??



2015-06-17 00:23:12 Error: [Cake\Error\FatalErrorException] syntax error, unexpected 'de' (T_STRING), expecting ']'
Request URL: /grupos/adicionar
Stack Trace:
#0 C:\Xampp\htdocs\florence-sistema\trunk\vendor\cakephp\cakephp\src\Error\BaseErrorHandler.php(90): Cake\Error\BaseErrorHandler->handleFatalError(4, 'syntax error, u...', 'C:\\Xampp\\htdocs...', 35)
#1 [internal function]: Cake\Error\BaseErrorHandler->Cake\Error\{closure}()
#2 {main}


2015-06-20 21:16:16 Error: Fatal Error (1): Maximum execution time of 30 seconds exceeded in [C:\Xampp\htdocs\florence-sistema\trunk\vendor\cakephp\cakephp\src\Database\SqliteCompiler.php, line 26]
Trace:
Cake\Error\BaseErrorHandler::handleFatalError() - CORE\src\Error\BaseErrorHandler.php, line 191
Cake\Error\BaseErrorHandler::Cake\Error\{closure}() - CORE\src\Error\BaseErrorHandler.php, line 90
[main] - [internal], line ??



2015-06-20 21:16:22 Error: [Cake\Routing\Exception\MissingControllerException] Controller class Florence-sistema could not be found.
Exception Attributes: array (
  'class' => 'Florence-sistema',
  'plugin' => NULL,
  'prefix' => NULL,
  '_ext' => NULL,
)
Request URL: /florence-sistema/trunk
Stack Trace:
#0 C:\Xampp\htdocs\florence-sistema\trunk\webroot\index.php(37): Cake\Routing\Dispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#1 {main}


2015-06-20 21:16:25 Error: [Cake\Routing\Exception\MissingControllerException] Controller class Florence-sistema could not be found.
Exception Attributes: array (
  'class' => 'Florence-sistema',
  'plugin' => NULL,
  'prefix' => NULL,
  '_ext' => NULL,
)
Request URL: /florence-sistema/trunk
Stack Trace:
#0 C:\Xampp\htdocs\florence-sistema\trunk\webroot\index.php(37): Cake\Routing\Dispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#1 {main}


2015-06-20 21:16:29 Error: [Cake\Routing\Exception\MissingControllerException] Controller class Favicon.ico could not be found.
Exception Attributes: array (
  'class' => 'Favicon.ico',
  'plugin' => NULL,
  'prefix' => NULL,
  '_ext' => NULL,
)
Request URL: /favicon.ico
Stack Trace:
#0 C:\Xampp\htdocs\florence-sistema\trunk\webroot\index.php(37): Cake\Routing\Dispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#1 {main}


2015-06-20 21:16:32 Error: [Cake\Routing\Exception\MissingControllerException] Controller class Favicon.ico could not be found.
Exception Attributes: array (
  'class' => 'Favicon.ico',
  'plugin' => NULL,
  'prefix' => NULL,
  '_ext' => NULL,
)
Request URL: /favicon.ico
Stack Trace:
#0 C:\Xampp\htdocs\florence-sistema\trunk\webroot\index.php(37): Cake\Routing\Dispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#1 {main}


2015-06-20 21:16:36 Error: [Cake\Routing\Exception\MissingControllerException] Controller class Favicon.ico could not be found.
Exception Attributes: array (
  'class' => 'Favicon.ico',
  'plugin' => NULL,
  'prefix' => NULL,
  '_ext' => NULL,
)
Request URL: /favicon.ico
Stack Trace:
#0 C:\Xampp\htdocs\florence-sistema\trunk\webroot\index.php(37): Cake\Routing\Dispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#1 {main}


2015-06-20 21:16:39 Error: [Cake\Routing\Exception\MissingControllerException] Controller class Favicon.ico could not be found.
Exception Attributes: array (
  'class' => 'Favicon.ico',
  'plugin' => NULL,
  'prefix' => NULL,
  '_ext' => NULL,
)
Request URL: /favicon.ico
Stack Trace:
#0 C:\Xampp\htdocs\florence-sistema\trunk\webroot\index.php(37): Cake\Routing\Dispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#1 {main}


2015-06-20 21:20:04 Error: [Cake\Routing\Exception\MissingControllerException] Controller class Onlinestore could not be found.
Exception Attributes: array (
  'class' => 'Onlinestore',
  'plugin' => NULL,
  'prefix' => NULL,
  '_ext' => NULL,
)
Request URL: /onlinestore/mixplayM.png
Stack Trace:
#0 C:\Xampp\htdocs\florence-sistema\trunk\webroot\index.php(37): Cake\Routing\Dispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#1 {main}


2015-06-20 22:03:49 Error: Fatal Error (4): syntax error, unexpected ';', expecting ']' in [C:\Xampp\htdocs\florence-sistema\trunk\src\Template\Agendas\index.ctp, line 56]
Trace:
Cake\Error\BaseErrorHandler::handleFatalError() - CORE\src\Error\BaseErrorHandler.php, line 191
Cake\Error\BaseErrorHandler::Cake\Error\{closure}() - CORE\src\Error\BaseErrorHandler.php, line 90
[main] - [internal], line ??



2015-06-20 22:03:50 Error: [Cake\Error\FatalErrorException] syntax error, unexpected ';', expecting ']'
Request URL: /agendas
Stack Trace:
#0 C:\Xampp\htdocs\florence-sistema\trunk\vendor\cakephp\cakephp\src\Error\BaseErrorHandler.php(90): Cake\Error\BaseErrorHandler->handleFatalError(4, 'syntax error, u...', 'C:\\Xampp\\htdocs...', 56)
#1 [internal function]: Cake\Error\BaseErrorHandler->Cake\Error\{closure}()
#2 {main}


2015-06-27 16:26:41 Error: [Cake\Controller\Exception\MissingActionException] Action ProntuariosController::anamnese() could not be found, or is not accessible.
Exception Attributes: array (
  'controller' => 'ProntuariosController',
  'action' => 'anamnese',
  'prefix' => '',
  'plugin' => NULL,
)
Request URL: /prontuarios/anamnese
Stack Trace:
#0 C:\Xampp\htdocs\florence-sistema\trunk\vendor\cakephp\cakephp\src\Routing\Dispatcher.php(114): Cake\Controller\Controller->invokeAction()
#1 C:\Xampp\htdocs\florence-sistema\trunk\vendor\cakephp\cakephp\src\Routing\Dispatcher.php(87): Cake\Routing\Dispatcher->_invoke(Object(App\Controller\ProntuariosController))
#2 C:\Xampp\htdocs\florence-sistema\trunk\webroot\index.php(37): Cake\Routing\Dispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#3 {main}


2015-06-27 16:29:08 Error: [Cake\Controller\Exception\MissingActionException] Action ProntuariosController::anamnese() could not be found, or is not accessible.
Exception Attributes: array (
  'controller' => 'ProntuariosController',
  'action' => 'anamnese',
  'prefix' => '',
  'plugin' => NULL,
)
Request URL: /prontuarios/anamnese
Stack Trace:
#0 C:\Xampp\htdocs\florence-sistema\trunk\vendor\cakephp\cakephp\src\Routing\Dispatcher.php(114): Cake\Controller\Controller->invokeAction()
#1 C:\Xampp\htdocs\florence-sistema\trunk\vendor\cakephp\cakephp\src\Routing\Dispatcher.php(87): Cake\Routing\Dispatcher->_invoke(Object(App\Controller\ProntuariosController))
#2 C:\Xampp\htdocs\florence-sistema\trunk\webroot\index.php(37): Cake\Routing\Dispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#3 {main}


2015-06-27 16:29:23 Error: [Cake\Controller\Exception\MissingActionException] Action ProntuariosController::diagnostico() could not be found, or is not accessible.
Exception Attributes: array (
  'controller' => 'ProntuariosController',
  'action' => 'diagnostico',
  'prefix' => '',
  'plugin' => NULL,
)
Request URL: /prontuarios/diagnostico
Stack Trace:
#0 C:\Xampp\htdocs\florence-sistema\trunk\vendor\cakephp\cakephp\src\Routing\Dispatcher.php(114): Cake\Controller\Controller->invokeAction()
#1 C:\Xampp\htdocs\florence-sistema\trunk\vendor\cakephp\cakephp\src\Routing\Dispatcher.php(87): Cake\Routing\Dispatcher->_invoke(Object(App\Controller\ProntuariosController))
#2 C:\Xampp\htdocs\florence-sistema\trunk\webroot\index.php(37): Cake\Routing\Dispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#3 {main}


2015-06-27 16:30:15 Error: [Cake\Controller\Exception\MissingActionException] Action ProntuariosController::anamnese() could not be found, or is not accessible.
Exception Attributes: array (
  'controller' => 'ProntuariosController',
  'action' => 'anamnese',
  'prefix' => '',
  'plugin' => NULL,
)
Request URL: /prontuarios/anamnese
Stack Trace:
#0 C:\Xampp\htdocs\florence-sistema\trunk\vendor\cakephp\cakephp\src\Routing\Dispatcher.php(114): Cake\Controller\Controller->invokeAction()
#1 C:\Xampp\htdocs\florence-sistema\trunk\vendor\cakephp\cakephp\src\Routing\Dispatcher.php(87): Cake\Routing\Dispatcher->_invoke(Object(App\Controller\ProntuariosController))
#2 C:\Xampp\htdocs\florence-sistema\trunk\webroot\index.php(37): Cake\Routing\Dispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#3 {main}


2015-06-27 16:30:36 Error: [Cake\Controller\Exception\MissingActionException] Action ProntuariosController::prescricao() could not be found, or is not accessible.
Exception Attributes: array (
  'controller' => 'ProntuariosController',
  'action' => 'prescricao',
  'prefix' => '',
  'plugin' => NULL,
)
Request URL: /prontuarios/prescricao
Stack Trace:
#0 C:\Xampp\htdocs\florence-sistema\trunk\vendor\cakephp\cakephp\src\Routing\Dispatcher.php(114): Cake\Controller\Controller->invokeAction()
#1 C:\Xampp\htdocs\florence-sistema\trunk\vendor\cakephp\cakephp\src\Routing\Dispatcher.php(87): Cake\Routing\Dispatcher->_invoke(Object(App\Controller\ProntuariosController))
#2 C:\Xampp\htdocs\florence-sistema\trunk\webroot\index.php(37): Cake\Routing\Dispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#3 {main}


2015-06-27 16:31:20 Error: [Cake\Controller\Exception\MissingActionException] Action ProntuariosController::receita() could not be found, or is not accessible.
Exception Attributes: array (
  'controller' => 'ProntuariosController',
  'action' => 'receita',
  'prefix' => '',
  'plugin' => NULL,
)
Request URL: /prontuarios/receita
Stack Trace:
#0 C:\Xampp\htdocs\florence-sistema\trunk\vendor\cakephp\cakephp\src\Routing\Dispatcher.php(114): Cake\Controller\Controller->invokeAction()
#1 C:\Xampp\htdocs\florence-sistema\trunk\vendor\cakephp\cakephp\src\Routing\Dispatcher.php(87): Cake\Routing\Dispatcher->_invoke(Object(App\Controller\ProntuariosController))
#2 C:\Xampp\htdocs\florence-sistema\trunk\webroot\index.php(37): Cake\Routing\Dispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#3 {main}


2015-06-27 16:33:08 Error: [Cake\Controller\Exception\MissingActionException] Action ProntuariosController::anamnese() could not be found, or is not accessible.
Exception Attributes: array (
  'controller' => 'ProntuariosController',
  'action' => 'anamnese',
  'prefix' => '',
  'plugin' => NULL,
)
Request URL: /prontuarios/anamnese
Stack Trace:
#0 C:\Xampp\htdocs\florence-sistema\trunk\vendor\cakephp\cakephp\src\Routing\Dispatcher.php(114): Cake\Controller\Controller->invokeAction()
#1 C:\Xampp\htdocs\florence-sistema\trunk\vendor\cakephp\cakephp\src\Routing\Dispatcher.php(87): Cake\Routing\Dispatcher->_invoke(Object(App\Controller\ProntuariosController))
#2 C:\Xampp\htdocs\florence-sistema\trunk\webroot\index.php(37): Cake\Routing\Dispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#3 {main}


2015-06-27 16:34:58 Error: [Cake\Controller\Exception\MissingActionException] Action ProntuariosController::anamnese() could not be found, or is not accessible.
Exception Attributes: array (
  'controller' => 'ProntuariosController',
  'action' => 'anamnese',
  'prefix' => '',
  'plugin' => NULL,
)
Request URL: /prontuarios/anamnese
Stack Trace:
#0 C:\Xampp\htdocs\florence-sistema\trunk\vendor\cakephp\cakephp\src\Routing\Dispatcher.php(114): Cake\Controller\Controller->invokeAction()
#1 C:\Xampp\htdocs\florence-sistema\trunk\vendor\cakephp\cakephp\src\Routing\Dispatcher.php(87): Cake\Routing\Dispatcher->_invoke(Object(App\Controller\ProntuariosController))
#2 C:\Xampp\htdocs\florence-sistema\trunk\webroot\index.php(37): Cake\Routing\Dispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#3 {main}


2015-06-27 16:36:30 Error: [Cake\Controller\Exception\MissingActionException] Action ProntuariosController::diagnostico() could not be found, or is not accessible.
Exception Attributes: array (
  'controller' => 'ProntuariosController',
  'action' => 'diagnostico',
  'prefix' => '',
  'plugin' => NULL,
)
Request URL: /prontuarios/diagnostico
Stack Trace:
#0 C:\Xampp\htdocs\florence-sistema\trunk\vendor\cakephp\cakephp\src\Routing\Dispatcher.php(114): Cake\Controller\Controller->invokeAction()
#1 C:\Xampp\htdocs\florence-sistema\trunk\vendor\cakephp\cakephp\src\Routing\Dispatcher.php(87): Cake\Routing\Dispatcher->_invoke(Object(App\Controller\ProntuariosController))
#2 C:\Xampp\htdocs\florence-sistema\trunk\webroot\index.php(37): Cake\Routing\Dispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#3 {main}


2015-06-27 16:52:12 Error: Fatal Error (4): syntax error, unexpected ',' in [C:\Xampp\htdocs\florence-sistema\trunk\src\Template\Prontuarios\adicionar.ctp, line 93]
Trace:
Cake\Error\BaseErrorHandler::handleFatalError() - CORE\src\Error\BaseErrorHandler.php, line 191
Cake\Error\BaseErrorHandler::Cake\Error\{closure}() - CORE\src\Error\BaseErrorHandler.php, line 90
[main] - [internal], line ??



2015-06-27 16:52:12 Error: [Cake\Error\FatalErrorException] syntax error, unexpected ','
Request URL: /prontuarios/adicionar
Stack Trace:
#0 C:\Xampp\htdocs\florence-sistema\trunk\vendor\cakephp\cakephp\src\Error\BaseErrorHandler.php(90): Cake\Error\BaseErrorHandler->handleFatalError(4, 'syntax error, u...', 'C:\\Xampp\\htdocs...', 93)
#1 [internal function]: Cake\Error\BaseErrorHandler->Cake\Error\{closure}()
#2 {main}


2015-07-09 00:31:19 Error: [Cake\Routing\Exception\MissingControllerException] Controller class Atendimentos could not be found.
Exception Attributes: array (
  'class' => 'Atendimentos',
  'plugin' => NULL,
  'prefix' => NULL,
  '_ext' => NULL,
)
Request URL: /atendimentos
Stack Trace:
#0 C:\Xampp\htdocs\florence-sistema\trunk\webroot\index.php(37): Cake\Routing\Dispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#1 {main}


2015-07-09 00:32:08 Error: [Cake\Routing\Exception\MissingControllerException] Controller class Atendimentos could not be found.
Exception Attributes: array (
  'class' => 'Atendimentos',
  'plugin' => NULL,
  'prefix' => NULL,
  '_ext' => NULL,
)
Request URL: /atendimentos
Stack Trace:
#0 C:\Xampp\htdocs\florence-sistema\trunk\webroot\index.php(37): Cake\Routing\Dispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#1 {main}


2015-07-09 01:27:37 Error: Fatal Error (4): syntax error, unexpected '1573' (T_LNUMBER) in [C:\Xampp\htdocs\florence-sistema\trunk\src\Controller\PacientesController.php, line 26]
Trace:
Cake\Error\BaseErrorHandler::handleFatalError() - CORE\src\Error\BaseErrorHandler.php, line 191
Cake\Error\BaseErrorHandler::Cake\Error\{closure}() - CORE\src\Error\BaseErrorHandler.php, line 90
[main] - [internal], line ??



2015-07-09 01:27:38 Error: [Cake\Error\FatalErrorException] syntax error, unexpected '1573' (T_LNUMBER)
Request URL: /pacientes/index
Stack Trace:
#0 C:\Xampp\htdocs\florence-sistema\trunk\vendor\cakephp\cakephp\src\Error\BaseErrorHandler.php(90): Cake\Error\BaseErrorHandler->handleFatalError(4, 'syntax error, u...', 'C:\\Xampp\\htdocs...', 26)
#1 [internal function]: Cake\Error\BaseErrorHandler->Cake\Error\{closure}()
#2 {main}


2015-07-09 01:50:11 Error: Fatal Error (1): Call to a member function get() on boolean in [C:\Xampp\htdocs\florence-sistema\trunk\src\Controller\PacientesController.php, line 12]
Trace:
Cake\Error\BaseErrorHandler::handleFatalError() - CORE\src\Error\BaseErrorHandler.php, line 191
Cake\Error\BaseErrorHandler::Cake\Error\{closure}() - CORE\src\Error\BaseErrorHandler.php, line 90
[main] - [internal], line ??



2015-07-09 01:50:12 Error: [Cake\Error\FatalErrorException] Call to a member function get() on boolean
Request URL: /pacientes/adicionar
Stack Trace:
#0 C:\Xampp\htdocs\florence-sistema\trunk\vendor\cakephp\cakephp\src\Error\BaseErrorHandler.php(90): Cake\Error\BaseErrorHandler->handleFatalError(1, 'Call to a membe...', 'C:\\Xampp\\htdocs...', 12)
#1 [internal function]: Cake\Error\BaseErrorHandler->Cake\Error\{closure}()
#2 {main}


2015-07-09 23:39:07 Warning: Warning (4096): Argument 1 passed to Cake\ORM\Table::save() must implement interface Cake\Datasource\EntityInterface, null given, called in C:\Xampp\htdocs\florence-sistema\trunk\src\Controller\PacientesController.php on line 15 and defined in [C:\Xampp\htdocs\florence-sistema\trunk\vendor\cakephp\cakephp\src\ORM\Table.php, line 1346]
Trace:
Cake\Error\BaseErrorHandler::handleError() - CORE\src\Error\BaseErrorHandler.php, line 138
Cake\ORM\Table::save() - CORE\src\ORM\Table.php, line 1346
App\Controller\PacientesController::adicionar() - APP/Controller\PacientesController.php, line 15
Cake\Controller\Controller::invokeAction() - CORE\src\Controller\Controller.php, line 404
Cake\Routing\Dispatcher::_invoke() - CORE\src\Routing\Dispatcher.php, line 114
Cake\Routing\Dispatcher::dispatch() - CORE\src\Routing\Dispatcher.php, line 87
[main] - ROOT\webroot\index.php, line 37



2015-07-09 23:39:08 Error: Fatal Error (1): Call to a member function errors() on null in [C:\Xampp\htdocs\florence-sistema\trunk\vendor\cakephp\cakephp\src\ORM\Table.php, line 1356]
Trace:
Cake\Error\BaseErrorHandler::handleFatalError() - CORE\src\Error\BaseErrorHandler.php, line 191
Cake\Error\BaseErrorHandler::Cake\Error\{closure}() - CORE\src\Error\BaseErrorHandler.php, line 90
[main] - [internal], line ??



2015-07-09 23:39:09 Error: [Cake\Error\FatalErrorException] Call to a member function errors() on null
Request URL: /pacientes/adicionar
Stack Trace:
#0 C:\Xampp\htdocs\florence-sistema\trunk\vendor\cakephp\cakephp\src\Error\BaseErrorHandler.php(90): Cake\Error\BaseErrorHandler->handleFatalError(1, 'Call to a membe...', 'C:\\Xampp\\htdocs...', 1356)
#1 [internal function]: Cake\Error\BaseErrorHandler->Cake\Error\{closure}()
#2 {main}


2015-07-09 23:53:11 Warning: Warning (4096): Argument 1 passed to Cake\ORM\Table::patchEntity() must be an instance of Cake\Datasource\EntityInterface, null given, called in C:\Xampp\htdocs\florence-sistema\trunk\src\Controller\PacientesController.php on line 14 and defined in [C:\Xampp\htdocs\florence-sistema\trunk\vendor\cakephp\cakephp\src\ORM\Table.php, line 2026]
Trace:
Cake\Error\BaseErrorHandler::handleError() - CORE\src\Error\BaseErrorHandler.php, line 138
Cake\ORM\Table::patchEntity() - CORE\src\ORM\Table.php, line 2026
App\Controller\PacientesController::adicionar() - APP/Controller\PacientesController.php, line 14
Cake\Controller\Controller::invokeAction() - CORE\src\Controller\Controller.php, line 404
Cake\Routing\Dispatcher::_invoke() - CORE\src\Routing\Dispatcher.php, line 114
Cake\Routing\Dispatcher::dispatch() - CORE\src\Routing\Dispatcher.php, line 87
[main] - ROOT\webroot\index.php, line 37



2015-07-09 23:53:11 Warning: Warning (4096): Argument 1 passed to Cake\ORM\Marshaller::merge() must be an instance of Cake\Datasource\EntityInterface, null given, called in C:\Xampp\htdocs\florence-sistema\trunk\vendor\cakephp\cakephp\src\ORM\Table.php on line 2032 and defined in [C:\Xampp\htdocs\florence-sistema\trunk\vendor\cakephp\cakephp\src\ORM\Marshaller.php, line 377]
Trace:
Cake\Error\BaseErrorHandler::handleError() - CORE\src\Error\BaseErrorHandler.php, line 138
Cake\ORM\Marshaller::merge() - CORE\src\ORM\Marshaller.php, line 377
Cake\ORM\Table::patchEntity() - CORE\src\ORM\Table.php, line 2032
App\Controller\PacientesController::adicionar() - APP/Controller\PacientesController.php, line 14
Cake\Controller\Controller::invokeAction() - CORE\src\Controller\Controller.php, line 404
Cake\Routing\Dispatcher::_invoke() - CORE\src\Routing\Dispatcher.php, line 114
Cake\Routing\Dispatcher::dispatch() - CORE\src\Routing\Dispatcher.php, line 87
[main] - ROOT\webroot\index.php, line 37



2015-07-09 23:53:11 Error: Fatal Error (1): Call to a member function isNew() on null in [C:\Xampp\htdocs\florence-sistema\trunk\vendor\cakephp\cakephp\src\ORM\Marshaller.php, line 382]
Trace:
Cake\Error\BaseErrorHandler::handleFatalError() - CORE\src\Error\BaseErrorHandler.php, line 191
Cake\Error\BaseErrorHandler::Cake\Error\{closure}() - CORE\src\Error\BaseErrorHandler.php, line 90
[main] - [internal], line ??



2015-07-09 23:53:12 Error: [Cake\Error\FatalErrorException] Call to a member function isNew() on null
Request URL: /pacientes/adicionar
Stack Trace:
#0 C:\Xampp\htdocs\florence-sistema\trunk\vendor\cakephp\cakephp\src\Error\BaseErrorHandler.php(90): Cake\Error\BaseErrorHandler->handleFatalError(1, 'Call to a membe...', 'C:\\Xampp\\htdocs...', 382)
#1 [internal function]: Cake\Error\BaseErrorHandler->Cake\Error\{closure}()
#2 {main}


2015-07-09 23:55:17 Error: Fatal Error (1): Call to a member function newEntity() on boolean in [C:\Xampp\htdocs\florence-sistema\trunk\src\Controller\PacientesController.php, line 12]
Trace:
Cake\Error\BaseErrorHandler::handleFatalError() - CORE\src\Error\BaseErrorHandler.php, line 191
Cake\Error\BaseErrorHandler::Cake\Error\{closure}() - CORE\src\Error\BaseErrorHandler.php, line 90
[main] - [internal], line ??



2015-07-09 23:55:18 Error: [Cake\Error\FatalErrorException] Call to a member function newEntity() on boolean
Request URL: /pacientes/adicionar
Stack Trace:
#0 C:\Xampp\htdocs\florence-sistema\trunk\vendor\cakephp\cakephp\src\Error\BaseErrorHandler.php(90): Cake\Error\BaseErrorHandler->handleFatalError(1, 'Call to a membe...', 'C:\\Xampp\\htdocs...', 12)
#1 [internal function]: Cake\Error\BaseErrorHandler->Cake\Error\{closure}()
#2 {main}


2015-07-10 00:02:30 Error: [RuntimeException] Table "Cake\ORM\Table" is not associated with "validationErrors"
Request URL: /pacientes/adicionar
Stack Trace:
#0 C:\Xampp\htdocs\florence-sistema\trunk\src\Controller\PacientesController.php(21): Cake\ORM\Table->__get('validationError...')
#1 [internal function]: App\Controller\PacientesController->adicionar()
#2 C:\Xampp\htdocs\florence-sistema\trunk\vendor\cakephp\cakephp\src\Controller\Controller.php(404): call_user_func_array(Array, Array)
#3 C:\Xampp\htdocs\florence-sistema\trunk\vendor\cakephp\cakephp\src\Routing\Dispatcher.php(114): Cake\Controller\Controller->invokeAction()
#4 C:\Xampp\htdocs\florence-sistema\trunk\vendor\cakephp\cakephp\src\Routing\Dispatcher.php(87): Cake\Routing\Dispatcher->_invoke(Object(App\Controller\PacientesController))
#5 C:\Xampp\htdocs\florence-sistema\trunk\webroot\index.php(37): Cake\Routing\Dispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#6 {main}


2015-07-10 00:19:48 Warning: Warning (4096): Argument 1 passed to Cake\ORM\Table::save() must be an instance of Cake\Datasource\EntityInterface, array given, called in C:\Xampp\htdocs\florence-sistema\trunk\src\Controller\PacientesController.php on line 18 and defined in [C:\Xampp\htdocs\florence-sistema\trunk\vendor\cakephp\cakephp\src\ORM\Table.php, line 1346]
Trace:
Cake\Error\BaseErrorHandler::handleError() - CORE\src\Error\BaseErrorHandler.php, line 138
Cake\ORM\Table::save() - CORE\src\ORM\Table.php, line 1346
App\Controller\PacientesController::adicionar() - APP/Controller\PacientesController.php, line 18
Cake\Controller\Controller::invokeAction() - CORE\src\Controller\Controller.php, line 404
Cake\Routing\Dispatcher::_invoke() - CORE\src\Routing\Dispatcher.php, line 114
Cake\Routing\Dispatcher::dispatch() - CORE\src\Routing\Dispatcher.php, line 87
[main] - ROOT\webroot\index.php, line 37



2015-07-10 00:19:48 Error: Fatal Error (1): Call to a member function errors() on array in [C:\Xampp\htdocs\florence-sistema\trunk\vendor\cakephp\cakephp\src\ORM\Table.php, line 1356]
Trace:
Cake\Error\BaseErrorHandler::handleFatalError() - CORE\src\Error\BaseErrorHandler.php, line 191
Cake\Error\BaseErrorHandler::Cake\Error\{closure}() - CORE\src\Error\BaseErrorHandler.php, line 90
[main] - [internal], line ??



2015-07-10 00:19:49 Error: [Cake\Error\FatalErrorException] Call to a member function errors() on array
Request URL: /pacientes/adicionar
Stack Trace:
#0 C:\Xampp\htdocs\florence-sistema\trunk\vendor\cakephp\cakephp\src\Error\BaseErrorHandler.php(90): Cake\Error\BaseErrorHandler->handleFatalError(1, 'Call to a membe...', 'C:\\Xampp\\htdocs...', 1356)
#1 [internal function]: Cake\Error\BaseErrorHandler->Cake\Error\{closure}()
#2 {main}


2015-07-10 00:20:21 Warning: Warning (4096): Argument 1 passed to Cake\ORM\Table::save() must implement interface Cake\Datasource\EntityInterface, array given, called in C:\Xampp\htdocs\florence-sistema\trunk\src\Controller\PacientesController.php on line 19 and defined in [C:\Xampp\htdocs\florence-sistema\trunk\vendor\cakephp\cakephp\src\ORM\Table.php, line 1346]
Trace:
Cake\Error\BaseErrorHandler::handleError() - CORE\src\Error\BaseErrorHandler.php, line 138
Cake\ORM\Table::save() - CORE\src\ORM\Table.php, line 1346
App\Controller\PacientesController::adicionar() - APP/Controller\PacientesController.php, line 19
Cake\Controller\Controller::invokeAction() - CORE\src\Controller\Controller.php, line 404
Cake\Routing\Dispatcher::_invoke() - CORE\src\Routing\Dispatcher.php, line 114
Cake\Routing\Dispatcher::dispatch() - CORE\src\Routing\Dispatcher.php, line 87
[main] - ROOT\webroot\index.php, line 37



2015-07-10 00:20:21 Error: Fatal Error (1): Call to a member function errors() on array in [C:\Xampp\htdocs\florence-sistema\trunk\vendor\cakephp\cakephp\src\ORM\Table.php, line 1356]
Trace:
Cake\Error\BaseErrorHandler::handleFatalError() - CORE\src\Error\BaseErrorHandler.php, line 191
Cake\Error\BaseErrorHandler::Cake\Error\{closure}() - CORE\src\Error\BaseErrorHandler.php, line 90
[main] - [internal], line ??



2015-07-10 00:20:22 Error: [Cake\Error\FatalErrorException] Call to a member function errors() on array
Request URL: /pacientes/adicionar
Stack Trace:
#0 C:\Xampp\htdocs\florence-sistema\trunk\vendor\cakephp\cakephp\src\Error\BaseErrorHandler.php(90): Cake\Error\BaseErrorHandler->handleFatalError(1, 'Call to a membe...', 'C:\\Xampp\\htdocs...', 1356)
#1 [internal function]: Cake\Error\BaseErrorHandler->Cake\Error\{closure}()
#2 {main}


2015-07-10 00:20:27 Warning: Warning (4096): Argument 1 passed to Cake\ORM\Table::save() must implement interface Cake\Datasource\EntityInterface, array given, called in C:\Xampp\htdocs\florence-sistema\trunk\src\Controller\PacientesController.php on line 19 and defined in [C:\Xampp\htdocs\florence-sistema\trunk\vendor\cakephp\cakephp\src\ORM\Table.php, line 1346]
Trace:
Cake\Error\BaseErrorHandler::handleError() - CORE\src\Error\BaseErrorHandler.php, line 138
Cake\ORM\Table::save() - CORE\src\ORM\Table.php, line 1346
App\Controller\PacientesController::adicionar() - APP/Controller\PacientesController.php, line 19
Cake\Controller\Controller::invokeAction() - CORE\src\Controller\Controller.php, line 404
Cake\Routing\Dispatcher::_invoke() - CORE\src\Routing\Dispatcher.php, line 114
Cake\Routing\Dispatcher::dispatch() - CORE\src\Routing\Dispatcher.php, line 87
[main] - ROOT\webroot\index.php, line 37



2015-07-10 00:20:27 Error: Fatal Error (1): Call to a member function errors() on array in [C:\Xampp\htdocs\florence-sistema\trunk\vendor\cakephp\cakephp\src\ORM\Table.php, line 1356]
Trace:
Cake\Error\BaseErrorHandler::handleFatalError() - CORE\src\Error\BaseErrorHandler.php, line 191
Cake\Error\BaseErrorHandler::Cake\Error\{closure}() - CORE\src\Error\BaseErrorHandler.php, line 90
[main] - [internal], line ??



2015-07-10 00:20:28 Error: [Cake\Error\FatalErrorException] Call to a member function errors() on array
Request URL: /pacientes/adicionar
Stack Trace:
#0 C:\Xampp\htdocs\florence-sistema\trunk\vendor\cakephp\cakephp\src\Error\BaseErrorHandler.php(90): Cake\Error\BaseErrorHandler->handleFatalError(1, 'Call to a membe...', 'C:\\Xampp\\htdocs...', 1356)
#1 [internal function]: Cake\Error\BaseErrorHandler->Cake\Error\{closure}()
#2 {main}


2015-07-10 00:32:20 Warning: Warning (4096): Argument 1 passed to Cake\ORM\Table::save() must be an instance of Cake\Datasource\EntityInterface, array given, called in C:\Xampp\htdocs\florence-sistema\trunk\src\Controller\PacientesController.php on line 19 and defined in [C:\Xampp\htdocs\florence-sistema\trunk\vendor\cakephp\cakephp\src\ORM\Table.php, line 1346]
Trace:
Cake\Error\BaseErrorHandler::handleError() - CORE\src\Error\BaseErrorHandler.php, line 138
Cake\ORM\Table::save() - CORE\src\ORM\Table.php, line 1346
App\Controller\PacientesController::adicionar() - APP/Controller\PacientesController.php, line 19
Cake\Controller\Controller::invokeAction() - CORE\src\Controller\Controller.php, line 404
Cake\Routing\Dispatcher::_invoke() - CORE\src\Routing\Dispatcher.php, line 114
Cake\Routing\Dispatcher::dispatch() - CORE\src\Routing\Dispatcher.php, line 87
[main] - ROOT\webroot\index.php, line 37



2015-07-10 00:32:20 Error: Fatal Error (1): Call to a member function errors() on array in [C:\Xampp\htdocs\florence-sistema\trunk\vendor\cakephp\cakephp\src\ORM\Table.php, line 1356]
Trace:
Cake\Error\BaseErrorHandler::handleFatalError() - CORE\src\Error\BaseErrorHandler.php, line 191
Cake\Error\BaseErrorHandler::Cake\Error\{closure}() - CORE\src\Error\BaseErrorHandler.php, line 90
[main] - [internal], line ??



2015-07-10 00:32:20 Error: [Cake\Error\FatalErrorException] Call to a member function errors() on array
Request URL: /pacientes/adicionar
Stack Trace:
#0 C:\Xampp\htdocs\florence-sistema\trunk\vendor\cakephp\cakephp\src\Error\BaseErrorHandler.php(90): Cake\Error\BaseErrorHandler->handleFatalError(1, 'Call to a membe...', 'C:\\Xampp\\htdocs...', 1356)
#1 [internal function]: Cake\Error\BaseErrorHandler->Cake\Error\{closure}()
#2 {main}


2015-07-10 00:33:01 Warning: Warning (4096): Argument 1 passed to Cake\ORM\Table::save() must be an instance of Cake\Datasource\EntityInterface, array given, called in C:\Xampp\htdocs\florence-sistema\trunk\src\Controller\PacientesController.php on line 19 and defined in [C:\Xampp\htdocs\florence-sistema\trunk\vendor\cakephp\cakephp\src\ORM\Table.php, line 1346]
Trace:
Cake\Error\BaseErrorHandler::handleError() - CORE\src\Error\BaseErrorHandler.php, line 138
Cake\ORM\Table::save() - CORE\src\ORM\Table.php, line 1346
App\Controller\PacientesController::adicionar() - APP/Controller\PacientesController.php, line 19
Cake\Controller\Controller::invokeAction() - CORE\src\Controller\Controller.php, line 404
Cake\Routing\Dispatcher::_invoke() - CORE\src\Routing\Dispatcher.php, line 114
Cake\Routing\Dispatcher::dispatch() - CORE\src\Routing\Dispatcher.php, line 87
[main] - ROOT\webroot\index.php, line 37



2015-07-10 00:33:01 Error: Fatal Error (1): Call to a member function errors() on array in [C:\Xampp\htdocs\florence-sistema\trunk\vendor\cakephp\cakephp\src\ORM\Table.php, line 1356]
Trace:
Cake\Error\BaseErrorHandler::handleFatalError() - CORE\src\Error\BaseErrorHandler.php, line 191
Cake\Error\BaseErrorHandler::Cake\Error\{closure}() - CORE\src\Error\BaseErrorHandler.php, line 90
[main] - [internal], line ??



2015-07-10 00:33:02 Error: [Cake\Error\FatalErrorException] Call to a member function errors() on array
Request URL: /pacientes/adicionar
Stack Trace:
#0 C:\Xampp\htdocs\florence-sistema\trunk\vendor\cakephp\cakephp\src\Error\BaseErrorHandler.php(90): Cake\Error\BaseErrorHandler->handleFatalError(1, 'Call to a membe...', 'C:\\Xampp\\htdocs...', 1356)
#1 [internal function]: Cake\Error\BaseErrorHandler->Cake\Error\{closure}()
#2 {main}


2015-07-10 00:33:38 Warning: Warning (4096): Argument 1 passed to Cake\ORM\Table::save() must implement interface Cake\Datasource\EntityInterface, none given, called in C:\Xampp\htdocs\florence-sistema\trunk\src\Controller\PacientesController.php on line 19 and defined in [C:\Xampp\htdocs\florence-sistema\trunk\vendor\cakephp\cakephp\src\ORM\Table.php, line 1346]
Trace:
Cake\Error\BaseErrorHandler::handleError() - CORE\src\Error\BaseErrorHandler.php, line 138
Cake\ORM\Table::save() - CORE\src\ORM\Table.php, line 1346
App\Controller\PacientesController::adicionar() - APP/Controller\PacientesController.php, line 19
Cake\Controller\Controller::invokeAction() - CORE\src\Controller\Controller.php, line 404
Cake\Routing\Dispatcher::_invoke() - CORE\src\Routing\Dispatcher.php, line 114
Cake\Routing\Dispatcher::dispatch() - CORE\src\Routing\Dispatcher.php, line 87
[main] - ROOT\webroot\index.php, line 37



2015-07-10 00:33:38 Error: Fatal Error (1): Call to a member function errors() on null in [C:\Xampp\htdocs\florence-sistema\trunk\vendor\cakephp\cakephp\src\ORM\Table.php, line 1356]
Trace:
Cake\Error\BaseErrorHandler::handleFatalError() - CORE\src\Error\BaseErrorHandler.php, line 191
Cake\Error\BaseErrorHandler::Cake\Error\{closure}() - CORE\src\Error\BaseErrorHandler.php, line 90
[main] - [internal], line ??



2015-07-10 00:33:38 Error: [Cake\Error\FatalErrorException] Call to a member function errors() on null
Request URL: /pacientes/adicionar
Stack Trace:
#0 C:\Xampp\htdocs\florence-sistema\trunk\vendor\cakephp\cakephp\src\Error\BaseErrorHandler.php(90): Cake\Error\BaseErrorHandler->handleFatalError(1, 'Call to a membe...', 'C:\\Xampp\\htdocs...', 1356)
#1 [internal function]: Cake\Error\BaseErrorHandler->Cake\Error\{closure}()
#2 {main}


2015-07-10 00:34:14 Warning: Warning (4096): Argument 1 passed to Cake\ORM\Table::save() must implement interface Cake\Datasource\EntityInterface, none given, called in C:\Xampp\htdocs\florence-sistema\trunk\src\Controller\PacientesController.php on line 16 and defined in [C:\Xampp\htdocs\florence-sistema\trunk\vendor\cakephp\cakephp\src\ORM\Table.php, line 1346]
Trace:
Cake\Error\BaseErrorHandler::handleError() - CORE\src\Error\BaseErrorHandler.php, line 138
Cake\ORM\Table::save() - CORE\src\ORM\Table.php, line 1346
App\Controller\PacientesController::adicionar() - APP/Controller\PacientesController.php, line 16
Cake\Controller\Controller::invokeAction() - CORE\src\Controller\Controller.php, line 404
Cake\Routing\Dispatcher::_invoke() - CORE\src\Routing\Dispatcher.php, line 114
Cake\Routing\Dispatcher::dispatch() - CORE\src\Routing\Dispatcher.php, line 87
[main] - ROOT\webroot\index.php, line 37



2015-07-10 00:34:14 Error: Fatal Error (1): Call to a member function errors() on null in [C:\Xampp\htdocs\florence-sistema\trunk\vendor\cakephp\cakephp\src\ORM\Table.php, line 1356]
Trace:
Cake\Error\BaseErrorHandler::handleFatalError() - CORE\src\Error\BaseErrorHandler.php, line 191
Cake\Error\BaseErrorHandler::Cake\Error\{closure}() - CORE\src\Error\BaseErrorHandler.php, line 90
[main] - [internal], line ??



2015-07-10 00:34:14 Error: [Cake\Error\FatalErrorException] Call to a member function errors() on null
Request URL: /pacientes/adicionar
Stack Trace:
#0 C:\Xampp\htdocs\florence-sistema\trunk\vendor\cakephp\cakephp\src\Error\BaseErrorHandler.php(90): Cake\Error\BaseErrorHandler->handleFatalError(1, 'Call to a membe...', 'C:\\Xampp\\htdocs...', 1356)
#1 [internal function]: Cake\Error\BaseErrorHandler->Cake\Error\{closure}()
#2 {main}


2015-07-10 00:39:14 Warning: Warning (4096): Argument 2 passed to Cake\ORM\Table::patchEntity() must be of the type array, none given, called in C:\Xampp\htdocs\florence-sistema\trunk\src\Controller\PacientesController.php on line 14 and defined in [C:\Xampp\htdocs\florence-sistema\trunk\vendor\cakephp\cakephp\src\ORM\Table.php, line 2026]
Trace:
Cake\Error\BaseErrorHandler::handleError() - CORE\src\Error\BaseErrorHandler.php, line 138
Cake\ORM\Table::patchEntity() - CORE\src\ORM\Table.php, line 2026
App\Controller\PacientesController::adicionar() - APP/Controller\PacientesController.php, line 14
Cake\Controller\Controller::invokeAction() - CORE\src\Controller\Controller.php, line 404
Cake\Routing\Dispatcher::_invoke() - CORE\src\Routing\Dispatcher.php, line 114
Cake\Routing\Dispatcher::dispatch() - CORE\src\Routing\Dispatcher.php, line 87
[main] - ROOT\webroot\index.php, line 37



2015-07-10 00:39:14 Warning: Warning (4096): Argument 2 passed to Cake\ORM\Marshaller::merge() must be of the type array, null given, called in C:\Xampp\htdocs\florence-sistema\trunk\vendor\cakephp\cakephp\src\ORM\Table.php on line 2032 and defined in [C:\Xampp\htdocs\florence-sistema\trunk\vendor\cakephp\cakephp\src\ORM\Marshaller.php, line 377]
Trace:
Cake\Error\BaseErrorHandler::handleError() - CORE\src\Error\BaseErrorHandler.php, line 138
Cake\ORM\Marshaller::merge() - CORE\src\ORM\Marshaller.php, line 377
Cake\ORM\Table::patchEntity() - CORE\src\ORM\Table.php, line 2032
App\Controller\PacientesController::adicionar() - APP/Controller\PacientesController.php, line 14
Cake\Controller\Controller::invokeAction() - CORE\src\Controller\Controller.php, line 404
Cake\Routing\Dispatcher::_invoke() - CORE\src\Routing\Dispatcher.php, line 114
Cake\Routing\Dispatcher::dispatch() - CORE\src\Routing\Dispatcher.php, line 87
[main] - ROOT\webroot\index.php, line 37



2015-07-10 00:39:15 Error: [InvalidArgumentException] Passed variable is not an array or object, using empty array instead
Request URL: /pacientes/adicionar
Stack Trace:
#0 C:\Xampp\htdocs\florence-sistema\trunk\vendor\cakephp\cakephp\src\ORM\Marshaller.php(200): ArrayObject->__construct(NULL)
#1 C:\Xampp\htdocs\florence-sistema\trunk\vendor\cakephp\cakephp\src\ORM\Marshaller.php(379): Cake\ORM\Marshaller->_prepareDataAndOptions(NULL, Array)
#2 C:\Xampp\htdocs\florence-sistema\trunk\vendor\cakephp\cakephp\src\ORM\Table.php(2032): Cake\ORM\Marshaller->merge(Object(Cake\ORM\Entity), NULL, Array)
#3 C:\Xampp\htdocs\florence-sistema\trunk\src\Controller\PacientesController.php(14): Cake\ORM\Table->patchEntity(Object(Cake\ORM\Entity))
#4 [internal function]: App\Controller\PacientesController->adicionar()
#5 C:\Xampp\htdocs\florence-sistema\trunk\vendor\cakephp\cakephp\src\Controller\Controller.php(404): call_user_func_array(Array, Array)
#6 C:\Xampp\htdocs\florence-sistema\trunk\vendor\cakephp\cakephp\src\Routing\Dispatcher.php(114): Cake\Controller\Controller->invokeAction()
#7 C:\Xampp\htdocs\florence-sistema\trunk\vendor\cakephp\cakephp\src\Routing\Dispatcher.php(87): Cake\Routing\Dispatcher->_invoke(Object(App\Controller\PacientesController))
#8 C:\Xampp\htdocs\florence-sistema\trunk\webroot\index.php(37): Cake\Routing\Dispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#9 {main}


2015-07-10 00:39:22 Warning: Warning (4096): Argument 2 passed to Cake\ORM\Table::patchEntity() must be of the type array, none given, called in C:\Xampp\htdocs\florence-sistema\trunk\src\Controller\PacientesController.php on line 14 and defined in [C:\Xampp\htdocs\florence-sistema\trunk\vendor\cakephp\cakephp\src\ORM\Table.php, line 2026]
Trace:
Cake\Error\BaseErrorHandler::handleError() - CORE\src\Error\BaseErrorHandler.php, line 138
Cake\ORM\Table::patchEntity() - CORE\src\ORM\Table.php, line 2026
App\Controller\PacientesController::adicionar() - APP/Controller\PacientesController.php, line 14
Cake\Controller\Controller::invokeAction() - CORE\src\Controller\Controller.php, line 404
Cake\Routing\Dispatcher::_invoke() - CORE\src\Routing\Dispatcher.php, line 114
Cake\Routing\Dispatcher::dispatch() - CORE\src\Routing\Dispatcher.php, line 87
[main] - ROOT\webroot\index.php, line 37



2015-07-10 00:39:22 Warning: Warning (4096): Argument 2 passed to Cake\ORM\Marshaller::merge() must be of the type array, null given, called in C:\Xampp\htdocs\florence-sistema\trunk\vendor\cakephp\cakephp\src\ORM\Table.php on line 2032 and defined in [C:\Xampp\htdocs\florence-sistema\trunk\vendor\cakephp\cakephp\src\ORM\Marshaller.php, line 377]
Trace:
Cake\Error\BaseErrorHandler::handleError() - CORE\src\Error\BaseErrorHandler.php, line 138
Cake\ORM\Marshaller::merge() - CORE\src\ORM\Marshaller.php, line 377
Cake\ORM\Table::patchEntity() - CORE\src\ORM\Table.php, line 2032
App\Controller\PacientesController::adicionar() - APP/Controller\PacientesController.php, line 14
Cake\Controller\Controller::invokeAction() - CORE\src\Controller\Controller.php, line 404
Cake\Routing\Dispatcher::_invoke() - CORE\src\Routing\Dispatcher.php, line 114
Cake\Routing\Dispatcher::dispatch() - CORE\src\Routing\Dispatcher.php, line 87
[main] - ROOT\webroot\index.php, line 37



2015-07-10 00:39:23 Error: [InvalidArgumentException] Passed variable is not an array or object, using empty array instead
Request URL: /pacientes/adicionar
Stack Trace:
#0 C:\Xampp\htdocs\florence-sistema\trunk\vendor\cakephp\cakephp\src\ORM\Marshaller.php(200): ArrayObject->__construct(NULL)
#1 C:\Xampp\htdocs\florence-sistema\trunk\vendor\cakephp\cakephp\src\ORM\Marshaller.php(379): Cake\ORM\Marshaller->_prepareDataAndOptions(NULL, Array)
#2 C:\Xampp\htdocs\florence-sistema\trunk\vendor\cakephp\cakephp\src\ORM\Table.php(2032): Cake\ORM\Marshaller->merge(Object(Cake\ORM\Entity), NULL, Array)
#3 C:\Xampp\htdocs\florence-sistema\trunk\src\Controller\PacientesController.php(14): Cake\ORM\Table->patchEntity(Object(Cake\ORM\Entity))
#4 [internal function]: App\Controller\PacientesController->adicionar()
#5 C:\Xampp\htdocs\florence-sistema\trunk\vendor\cakephp\cakephp\src\Controller\Controller.php(404): call_user_func_array(Array, Array)
#6 C:\Xampp\htdocs\florence-sistema\trunk\vendor\cakephp\cakephp\src\Routing\Dispatcher.php(114): Cake\Controller\Controller->invokeAction()
#7 C:\Xampp\htdocs\florence-sistema\trunk\vendor\cakephp\cakephp\src\Routing\Dispatcher.php(87): Cake\Routing\Dispatcher->_invoke(Object(App\Controller\PacientesController))
#8 C:\Xampp\htdocs\florence-sistema\trunk\webroot\index.php(37): Cake\Routing\Dispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#9 {main}


2015-07-10 00:45:07 Warning: Warning (4096): Argument 1 passed to Cake\ORM\Table::patchEntity() must be an instance of Cake\Datasource\EntityInterface, null given, called in C:\Xampp\htdocs\florence-sistema\trunk\src\Controller\PacientesController.php on line 15 and defined in [C:\Xampp\htdocs\florence-sistema\trunk\vendor\cakephp\cakephp\src\ORM\Table.php, line 2026]
Trace:
Cake\Error\BaseErrorHandler::handleError() - CORE\src\Error\BaseErrorHandler.php, line 138
Cake\ORM\Table::patchEntity() - CORE\src\ORM\Table.php, line 2026
App\Controller\PacientesController::adicionar() - APP/Controller\PacientesController.php, line 15
Cake\Controller\Controller::invokeAction() - CORE\src\Controller\Controller.php, line 404
Cake\Routing\Dispatcher::_invoke() - CORE\src\Routing\Dispatcher.php, line 114
Cake\Routing\Dispatcher::dispatch() - CORE\src\Routing\Dispatcher.php, line 87
[main] - ROOT\webroot\index.php, line 37



2015-07-10 00:45:07 Warning: Warning (4096): Argument 1 passed to Cake\ORM\Marshaller::merge() must be an instance of Cake\Datasource\EntityInterface, null given, called in C:\Xampp\htdocs\florence-sistema\trunk\vendor\cakephp\cakephp\src\ORM\Table.php on line 2032 and defined in [C:\Xampp\htdocs\florence-sistema\trunk\vendor\cakephp\cakephp\src\ORM\Marshaller.php, line 377]
Trace:
Cake\Error\BaseErrorHandler::handleError() - CORE\src\Error\BaseErrorHandler.php, line 138
Cake\ORM\Marshaller::merge() - CORE\src\ORM\Marshaller.php, line 377
Cake\ORM\Table::patchEntity() - CORE\src\ORM\Table.php, line 2032
App\Controller\PacientesController::adicionar() - APP/Controller\PacientesController.php, line 15
Cake\Controller\Controller::invokeAction() - CORE\src\Controller\Controller.php, line 404
Cake\Routing\Dispatcher::_invoke() - CORE\src\Routing\Dispatcher.php, line 114
Cake\Routing\Dispatcher::dispatch() - CORE\src\Routing\Dispatcher.php, line 87
[main] - ROOT\webroot\index.php, line 37



2015-07-10 00:45:07 Error: Fatal Error (1): Call to a member function isNew() on null in [C:\Xampp\htdocs\florence-sistema\trunk\vendor\cakephp\cakephp\src\ORM\Marshaller.php, line 382]
Trace:
Cake\Error\BaseErrorHandler::handleFatalError() - CORE\src\Error\BaseErrorHandler.php, line 191
Cake\Error\BaseErrorHandler::Cake\Error\{closure}() - CORE\src\Error\BaseErrorHandler.php, line 90
[main] - [internal], line ??



2015-07-10 00:45:08 Error: [Cake\Error\FatalErrorException] Call to a member function isNew() on null
Request URL: /pacientes/adicionar
Stack Trace:
#0 C:\Xampp\htdocs\florence-sistema\trunk\vendor\cakephp\cakephp\src\Error\BaseErrorHandler.php(90): Cake\Error\BaseErrorHandler->handleFatalError(1, 'Call to a membe...', 'C:\\Xampp\\htdocs...', 382)
#1 [internal function]: Cake\Error\BaseErrorHandler->Cake\Error\{closure}()
#2 {main}


2015-07-10 00:49:46 Error: Fatal Error (1): Call to a member function validator() on boolean in [C:\Xampp\htdocs\florence-sistema\trunk\src\Controller\PacientesController.php, line 13]
Trace:
Cake\Error\BaseErrorHandler::handleFatalError() - CORE\src\Error\BaseErrorHandler.php, line 191
Cake\Error\BaseErrorHandler::Cake\Error\{closure}() - CORE\src\Error\BaseErrorHandler.php, line 90
[main] - [internal], line ??



2015-07-10 00:49:47 Error: [Cake\Error\FatalErrorException] Call to a member function validator() on boolean
Request URL: /pacientes/adicionar
Stack Trace:
#0 C:\Xampp\htdocs\florence-sistema\trunk\vendor\cakephp\cakephp\src\Error\BaseErrorHandler.php(90): Cake\Error\BaseErrorHandler->handleFatalError(1, 'Call to a membe...', 'C:\\Xampp\\htdocs...', 13)
#1 [internal function]: Cake\Error\BaseErrorHandler->Cake\Error\{closure}()
#2 {main}


2015-07-10 01:13:11 Error: Fatal Error (4): syntax error, unexpected '*' in [C:\Xampp\htdocs\florence-sistema\trunk\src\Model\Table\PacientesTable.php, line 22]
Trace:
Cake\Error\BaseErrorHandler::handleFatalError() - CORE\src\Error\BaseErrorHandler.php, line 191
Cake\Error\BaseErrorHandler::Cake\Error\{closure}() - CORE\src\Error\BaseErrorHandler.php, line 90
[main] - [internal], line ??



2015-07-10 01:13:12 Error: [Cake\Error\FatalErrorException] syntax error, unexpected '*'
Request URL: /pacientes/adicionar
Stack Trace:
#0 C:\Xampp\htdocs\florence-sistema\trunk\vendor\cakephp\cakephp\src\Error\BaseErrorHandler.php(90): Cake\Error\BaseErrorHandler->handleFatalError(4, 'syntax error, u...', 'C:\\Xampp\\htdocs...', 22)
#1 [internal function]: Cake\Error\BaseErrorHandler->Cake\Error\{closure}()
#2 {main}


2015-07-10 01:16:23 Error: Fatal Error (4): syntax error, unexpected '->' (T_OBJECT_OPERATOR) in [C:\Xampp\htdocs\florence-sistema\trunk\src\Model\Table\PacientesTable.php, line 14]
Trace:
Cake\Error\BaseErrorHandler::handleFatalError() - CORE\src\Error\BaseErrorHandler.php, line 191
Cake\Error\BaseErrorHandler::Cake\Error\{closure}() - CORE\src\Error\BaseErrorHandler.php, line 90
[main] - [internal], line ??



2015-07-10 01:16:25 Error: [Cake\Error\FatalErrorException] syntax error, unexpected '->' (T_OBJECT_OPERATOR)
Request URL: /pacientes/adicionar
Stack Trace:
#0 C:\Xampp\htdocs\florence-sistema\trunk\vendor\cakephp\cakephp\src\Error\BaseErrorHandler.php(90): Cake\Error\BaseErrorHandler->handleFatalError(4, 'syntax error, u...', 'C:\\Xampp\\htdocs...', 14)
#1 [internal function]: Cake\Error\BaseErrorHandler->Cake\Error\{closure}()
#2 {main}


2015-07-10 01:18:15 Error: Fatal Error (4): syntax error, unexpected 'public' (T_PUBLIC) in [C:\Xampp\htdocs\florence-sistema\trunk\src\Controller\PacientesController.php, line 32]
Trace:
Cake\Error\BaseErrorHandler::handleFatalError() - CORE\src\Error\BaseErrorHandler.php, line 191
Cake\Error\BaseErrorHandler::Cake\Error\{closure}() - CORE\src\Error\BaseErrorHandler.php, line 90
[main] - [internal], line ??



2015-07-10 01:18:16 Error: [Cake\Error\FatalErrorException] syntax error, unexpected 'public' (T_PUBLIC)
Request URL: /pacientes/adicionar
Stack Trace:
#0 C:\Xampp\htdocs\florence-sistema\trunk\vendor\cakephp\cakephp\src\Error\BaseErrorHandler.php(90): Cake\Error\BaseErrorHandler->handleFatalError(4, 'syntax error, u...', 'C:\\Xampp\\htdocs...', 32)
#1 [internal function]: Cake\Error\BaseErrorHandler->Cake\Error\{closure}()
#2 {main}


2015-07-10 01:19:30 Warning: Warning (2): Missing argument 2 for Cake\Validation\Validator::add(), called in C:\Xampp\htdocs\florence-sistema\trunk\src\Model\Table\PacientesTable.php on line 16 and defined in [C:\Xampp\htdocs\florence-sistema\trunk\vendor\cakephp\cakephp\src\Validation\Validator.php, line 291]
Trace:
Cake\Error\BaseErrorHandler::handleError() - CORE\src\Error\BaseErrorHandler.php, line 138
Cake\Validation\Validator::add() - CORE\src\Validation\Validator.php, line 291
App\Model\Table\PacientesTable::validationDefault() - APP/Model\Table\PacientesTable.php, line 16
Cake\ORM\Table::validator() - CORE\src\ORM\Table.php, line 1214
Cake\ORM\Marshaller::_validate() - CORE\src\ORM\Marshaller.php, line 170
Cake\ORM\Marshaller::one() - CORE\src\ORM\Marshaller.php, line 119
Cake\ORM\Table::newEntity() - CORE\src\ORM\Table.php, line 1955
App\Controller\PacientesController::adicionar() - APP/Controller\PacientesController.php, line 14
Cake\Controller\Controller::invokeAction() - CORE\src\Controller\Controller.php, line 404
Cake\Routing\Dispatcher::_invoke() - CORE\src\Routing\Dispatcher.php, line 114
Cake\Routing\Dispatcher::dispatch() - CORE\src\Routing\Dispatcher.php, line 87
[main] - ROOT\webroot\index.php, line 37



2015-07-10 01:42:08 Error: Fatal Error (1): Call to undefined function App\Controller\newEntity() in [C:\Xampp\htdocs\florence-sistema\trunk\src\Controller\PacientesController.php, line 20]
Trace:
Cake\Error\BaseErrorHandler::handleFatalError() - CORE\src\Error\BaseErrorHandler.php, line 191
Cake\Error\BaseErrorHandler::Cake\Error\{closure}() - CORE\src\Error\BaseErrorHandler.php, line 90
[main] - [internal], line ??



2015-07-10 01:42:09 Error: [Cake\Error\FatalErrorException] Call to undefined function App\Controller\newEntity()
Request URL: /pacientes/adicionar
Stack Trace:
#0 C:\Xampp\htdocs\florence-sistema\trunk\vendor\cakephp\cakephp\src\Error\BaseErrorHandler.php(90): Cake\Error\BaseErrorHandler->handleFatalError(1, 'Call to undefin...', 'C:\\Xampp\\htdocs...', 20)
#1 [internal function]: Cake\Error\BaseErrorHandler->Cake\Error\{closure}()
#2 {main}


2015-07-10 01:46:43 Error: Fatal Error (4): syntax error, unexpected '$validator' (T_VARIABLE) in [C:\Xampp\htdocs\florence-sistema\trunk\src\Model\Table\PacientesTable.php, line 14]
Trace:
Cake\Error\BaseErrorHandler::handleFatalError() - CORE\src\Error\BaseErrorHandler.php, line 191
Cake\Error\BaseErrorHandler::Cake\Error\{closure}() - CORE\src\Error\BaseErrorHandler.php, line 90
[main] - [internal], line ??



2015-07-10 01:46:44 Error: [Cake\Error\FatalErrorException] syntax error, unexpected '$validator' (T_VARIABLE)
Request URL: /pacientes/adicionar
Stack Trace:
#0 C:\Xampp\htdocs\florence-sistema\trunk\vendor\cakephp\cakephp\src\Error\BaseErrorHandler.php(90): Cake\Error\BaseErrorHandler->handleFatalError(4, 'syntax error, u...', 'C:\\Xampp\\htdocs...', 14)
#1 [internal function]: Cake\Error\BaseErrorHandler->Cake\Error\{closure}()
#2 {main}


2015-07-10 01:47:29 Error: Fatal Error (4): syntax error, unexpected 'return' (T_RETURN) in [C:\Xampp\htdocs\florence-sistema\trunk\src\Model\Table\PacientesTable.php, line 18]
Trace:
Cake\Error\BaseErrorHandler::handleFatalError() - CORE\src\Error\BaseErrorHandler.php, line 191
Cake\Error\BaseErrorHandler::Cake\Error\{closure}() - CORE\src\Error\BaseErrorHandler.php, line 90
[main] - [internal], line ??



2015-07-10 01:47:30 Error: [Cake\Error\FatalErrorException] syntax error, unexpected 'return' (T_RETURN)
Request URL: /pacientes/adicionar
Stack Trace:
#0 C:\Xampp\htdocs\florence-sistema\trunk\vendor\cakephp\cakephp\src\Error\BaseErrorHandler.php(90): Cake\Error\BaseErrorHandler->handleFatalError(4, 'syntax error, u...', 'C:\\Xampp\\htdocs...', 18)
#1 [internal function]: Cake\Error\BaseErrorHandler->Cake\Error\{closure}()
#2 {main}


2015-07-10 01:48:45 Error: Fatal Error (4): syntax error, unexpected '->' (T_OBJECT_OPERATOR) in [C:\Xampp\htdocs\florence-sistema\trunk\src\Model\Table\PacientesTable.php, line 13]
Trace:
Cake\Error\BaseErrorHandler::handleFatalError() - CORE\src\Error\BaseErrorHandler.php, line 191
Cake\Error\BaseErrorHandler::Cake\Error\{closure}() - CORE\src\Error\BaseErrorHandler.php, line 90
[main] - [internal], line ??



2015-07-10 01:48:46 Error: [Cake\Error\FatalErrorException] syntax error, unexpected '->' (T_OBJECT_OPERATOR)
Request URL: /pacientes/adicionar
Stack Trace:
#0 C:\Xampp\htdocs\florence-sistema\trunk\vendor\cakephp\cakephp\src\Error\BaseErrorHandler.php(90): Cake\Error\BaseErrorHandler->handleFatalError(4, 'syntax error, u...', 'C:\\Xampp\\htdocs...', 13)
#1 [internal function]: Cake\Error\BaseErrorHandler->Cake\Error\{closure}()
#2 {main}


2015-07-12 19:29:41 Error: [Cake\Routing\Exception\MissingControllerException] Controller class Webserver could not be found.
Exception Attributes: array (
  'class' => 'Webserver',
  'plugin' => NULL,
  'prefix' => NULL,
  '_ext' => NULL,
)
Request URL: /webserver/servidor.php
Stack Trace:
#0 C:\xampp\htdocs\florence-sistema\trunk\webroot\index.php(37): Cake\Routing\Dispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#1 {main}


2015-07-12 19:29:44 Error: [Cake\Routing\Exception\MissingControllerException] Controller class Webserver could not be found.
Exception Attributes: array (
  'class' => 'Webserver',
  'plugin' => NULL,
  'prefix' => NULL,
  '_ext' => NULL,
)
Request URL: /webserver/servidor.php
Stack Trace:
#0 C:\xampp\htdocs\florence-sistema\trunk\webroot\index.php(37): Cake\Routing\Dispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#1 {main}


2015-07-12 19:32:26 Error: [Cake\Routing\Exception\MissingControllerException] Controller class Webserver could not be found.
Exception Attributes: array (
  'class' => 'Webserver',
  'plugin' => NULL,
  'prefix' => NULL,
  '_ext' => NULL,
)
Request URL: /webserver/servidor.php
Stack Trace:
#0 C:\xampp\htdocs\florence-sistema\trunk\webroot\index.php(37): Cake\Routing\Dispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#1 {main}


2015-07-12 19:32:36 Error: [Cake\Routing\Exception\MissingControllerException] Controller class Webserver could not be found.
Exception Attributes: array (
  'class' => 'Webserver',
  'plugin' => NULL,
  'prefix' => NULL,
  '_ext' => NULL,
)
Request URL: /webserver/servidorWebService.php
Stack Trace:
#0 C:\xampp\htdocs\florence-sistema\trunk\webroot\index.php(37): Cake\Routing\Dispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#1 {main}


2015-07-12 19:37:42 Error: [Cake\Routing\Exception\MissingControllerException] Controller class Webserver could not be found.
Exception Attributes: array (
  'class' => 'Webserver',
  'plugin' => NULL,
  'prefix' => NULL,
  '_ext' => NULL,
)
Request URL: /webserver/servidorWebService.php
Stack Trace:
#0 C:\xampp\htdocs\florence-sistema\trunk\webroot\index.php(37): Cake\Routing\Dispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#1 {main}


2015-07-12 19:45:22 Error: [Cake\Routing\Exception\MissingControllerException] Controller class Webserver could not be found.
Exception Attributes: array (
  'class' => 'Webserver',
  'plugin' => NULL,
  'prefix' => NULL,
  '_ext' => NULL,
)
Request URL: /webserver/webservice.php
Stack Trace:
#0 C:\xampp\htdocs\florence-sistema\trunk\webroot\index.php(37): Cake\Routing\Dispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#1 {main}


2015-07-13 23:08:17 Error: [Cake\Routing\Exception\MissingControllerException] Controller class Vendor could not be found.
Exception Attributes: array (
  'class' => 'Vendor',
  'plugin' => NULL,
  'prefix' => NULL,
  '_ext' => NULL,
)
Request URL: /vendor/webserver/
Stack Trace:
#0 C:\xampp\htdocs\florence-sistema\trunk\webroot\index.php(37): Cake\Routing\Dispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#1 {main}


2015-07-13 23:12:06 Error: [Cake\Routing\Exception\MissingControllerException] Controller class Vendor could not be found.
Exception Attributes: array (
  'class' => 'Vendor',
  'plugin' => NULL,
  'prefix' => NULL,
  '_ext' => NULL,
)
Request URL: /vendor/webserver/server.php
Stack Trace:
#0 C:\xampp\htdocs\florence-sistema\trunk\webroot\index.php(37): Cake\Routing\Dispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#1 {main}


2015-07-13 23:12:10 Error: [Cake\Routing\Exception\MissingControllerException] Controller class Vendor could not be found.
Exception Attributes: array (
  'class' => 'Vendor',
  'plugin' => NULL,
  'prefix' => NULL,
  '_ext' => NULL,
)
Request URL: /vendor/webserver/server.php
Stack Trace:
#0 C:\xampp\htdocs\florence-sistema\trunk\webroot\index.php(37): Cake\Routing\Dispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#1 {main}


2015-07-13 23:12:27 Error: [Cake\Routing\Exception\MissingControllerException] Controller class Vendor could not be found.
Exception Attributes: array (
  'class' => 'Vendor',
  'plugin' => NULL,
  'prefix' => NULL,
  '_ext' => NULL,
)
Request URL: /vendor/webserver/webservice.php
Stack Trace:
#0 C:\xampp\htdocs\florence-sistema\trunk\webroot\index.php(37): Cake\Routing\Dispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#1 {main}


2015-07-13 23:13:23 Error: [Cake\Routing\Exception\MissingControllerException] Controller class Vendor could not be found.
Exception Attributes: array (
  'class' => 'Vendor',
  'plugin' => NULL,
  'prefix' => NULL,
  '_ext' => NULL,
)
Request URL: /vendor/webserver/wwebservice.php
Stack Trace:
#0 C:\xampp\htdocs\florence-sistema\trunk\webroot\index.php(37): Cake\Routing\Dispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#1 {main}


2015-07-13 23:13:31 Error: [Cake\Routing\Exception\MissingControllerException] Controller class Vendor could not be found.
Exception Attributes: array (
  'class' => 'Vendor',
  'plugin' => NULL,
  'prefix' => NULL,
  '_ext' => NULL,
)
Request URL: /vendor/webserver/webservice.php
Stack Trace:
#0 C:\xampp\htdocs\florence-sistema\trunk\webroot\index.php(37): Cake\Routing\Dispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#1 {main}


2015-07-13 23:19:23 Error: [Cake\Routing\Exception\MissingControllerException] Controller class Vendor could not be found.
Exception Attributes: array (
  'class' => 'Vendor',
  'plugin' => NULL,
  'prefix' => NULL,
  '_ext' => NULL,
)
Request URL: /vendor/webserver/webservice.php
Stack Trace:
#0 C:\xampp\htdocs\florence-sistema\trunk\webroot\index.php(37): Cake\Routing\Dispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#1 {main}


